{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Notebook based on original code and data from Gutierrez et al. 2020 https://doi.org/10.1038/s41467-019-13867-y."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Load libraries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import cobra\n",
    "import pdb\n",
    "import cobra_io_mat # modified cobrapy functions to preserve more model fields when exporting to matlab format"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Read files necessary to expand recon2.2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "f = open(\"rxnFormula_HUMAN.txt\")\n",
    "rxnFormula = f.read().splitlines()\n",
    "f.close()\n",
    "\n",
    "f = open(\"rxnAbbreviation_HUMAN.txt\")\n",
    "rxnAbbreviation = f.read().splitlines()\n",
    "f.close()\n",
    "\n",
    "f = open(\"rxnPathway_HUMAN.txt\")\n",
    "rxnPathway = f.read().splitlines()\n",
    "f.close()\n",
    "\n",
    "f = open(\"rxnConditions_HUMAN.txt\")\n",
    "rxnConditions = f.read().splitlines()\n",
    "f.close()\n",
    "\n",
    "f = open(\"rxnGPRs_HUMAN_HGNC.txt\")\n",
    "rxnGPR = f.read().splitlines()\n",
    "f.close()\n",
    "\n",
    "# Load PSIM matrix\n",
    "f = open('PSIM_HUMAN.tab','r')\n",
    "PSIM=f.read().splitlines()\n",
    "f.close()\n",
    "\n",
    "# Extract the Uniprot IDs\n",
    "PSIM_entries = []\n",
    "for line in PSIM:\n",
    "    PSIM_entries.append(line.split('\\t')[0])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Define Basic Functions"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Define a functions that counts of amino acids\n",
    "def count_AAs(sequence):\n",
    "    AAcounts = []    \n",
    "    AAs = ['G', 'A', 'V', 'L', 'I', 'M', 'W', 'F', 'P', 'S', 'T', 'C', 'Y', 'N', 'Q', 'E', 'D', 'K', 'R', 'H']\n",
    "    for aa in AAs:\n",
    "        AAcounts.append(sequence.count(aa))\n",
    "    return AAcounts\n",
    "\n",
    "#Define a function that substitutes question marks for AA counts in a formula (string)\n",
    "#Useful for translation and protein degradation pathway\n",
    "\n",
    "def substitute_AAs_count(formula,AAcounts):\n",
    "    template = \"? gly[c] + ? ala_L[c] + ? val_L[c] + ? leu_L[c] + ? ile_L[c] + ? met_L[c] + ? trp_L[c] + ? phe_L[c] + ? pro_L[c] + ? ser_L[c] + ? thr_L[c] + ? cys_L[c] + ? tyr_L[c] + ? asn_L[c] + ? gln_L[c] + ? glu_L[c] + ? asp_L[c] + ? lys_L[c] + ? arg_L[c] + ? his_L[c]\"\n",
    "    new = ''\n",
    "    index = 0\n",
    "    for character in template:\n",
    "        if character == \"?\":\n",
    "            new = new + str(AAcounts[index])\n",
    "            index = index + 1\n",
    "        else:\n",
    "            new = new + character\n",
    "    newFormula = formula.replace(template,new)\n",
    "    return newFormula\n",
    "    \n",
    "# Function that generates the translation reaction of a protein given its UniProt ID\n",
    "def translate_protein(entryID):\n",
    "    #Obtain protein sequence and length\n",
    "    PSI_row = PSIM[PSIM_entries.index(str(entryID))] \n",
    "    PSI_row = PSI_row.split('\\t')\n",
    "    sequence = PSI_row[11]\n",
    "    AAcounts = count_AAs(sequence)\n",
    "    N = len(sequence) #Number to replace atp's, gtp's, ppi's, pi's, h2o's, amp's, and gpd's\n",
    "    templateFormula = \"? h2o[c] + ? atp[c] + ? gtp[c] + ? gly[c] + ? ala_L[c] + ? val_L[c] + ? leu_L[c] + ? ile_L[c] + ? met_L[c] + ? trp_L[c] + ? phe_L[c] + ? pro_L[c] + ? ser_L[c] + ? thr_L[c] + ? cys_L[c] + ? tyr_L[c] + ? asn_L[c] + ? gln_L[c] + ? glu_L[c] + ? asp_L[c] + ? lys_L[c] + ? arg_L[c] + ? his_L[c] -> ? h[c] + ? amp[c] + adp[c] + ? pi[c] + ? gdp[c] + ? ppi[c] + XXX[c]\"\n",
    "    translationFormula = substitute_AAs_count(templateFormula, AAcounts)\n",
    "    translationFormula = translationFormula.replace(\"? h2o[c]\",str(2*N-1)+\" h2o[c]\")\n",
    "    translationFormula = translationFormula.replace(\"? h[c]\",str(2*N-1)+\" h[c]\")\n",
    "    translationFormula = translationFormula.replace(\"? ppi[c]\",str(N)+\" ppi[c]\")\n",
    "    translationFormula = translationFormula.replace(\"? pi[c]\",str(2*N-1)+\" pi[c]\")\n",
    "    translationFormula = translationFormula.replace(\"? gdp[c]\",str(2*N-2)+\" gdp[c]\")\n",
    "    translationFormula = translationFormula.replace(\"? amp[c]\",str(N)+\" amp[c]\")\n",
    "    translationFormula = translationFormula.replace(\"? gtp[c]\",str(2*N-2)+\" gtp[c]\")\n",
    "    translationFormula = translationFormula.replace(\"? atp[c]\",str(N+1)+\" atp[c]\")\n",
    "    translationFormula = translationFormula.replace(\"XXX[c]\",str(entryID)+\"[c]\")\n",
    "    return translationFormula\n",
    "\n",
    "# Function that replaces the XXX template name for the UniProt ID\n",
    "def insert_prot_name_in_rxnFormula(formula,entryID):\n",
    "    newFormula = formula.replace(\"XXX\",str(entryID))\n",
    "    return newFormula\n",
    "\n",
    "# Function that replaces the XXX template name for the Reaction abbreviation\n",
    "def insert_prot_name_in_rxnName(rxnAbbrev,entryID):\n",
    "    newAbbreviation = str(entryID) + \"_\" + rxnAbbrev\n",
    "    return newAbbreviation\n",
    "\n",
    "#Function for adding the reactions of a given PathwayName\n",
    "def addPathway(pathwayName,listOfRxns,listOfRxnsNames,listOfPathways):   \n",
    "    newList = listOfRxns    \n",
    "    newList2 = listOfRxnsNames\n",
    "    newList3 = listOfPathways\n",
    "    for i in range(len(rxnPathway)):\n",
    "        if rxnPathway[i] == pathwayName:\n",
    "            newList.append(rxnFormula[i])\n",
    "            newList2.append(rxnAbbreviation[i])\n",
    "            newList3.append(pathwayName)\n",
    "    return newList,newList2,newList3\n",
    "\n",
    "#Function for adding the reactions of a given PathwayName given a condition\n",
    "def addPathwayFromCondition(conditionName,listOfRxns,listOfRxnsNames,listOfPathways):   \n",
    "    newList = listOfRxns \n",
    "    newList2 = listOfRxnsNames\n",
    "    newList3 = listOfPathways\n",
    "    for i in range(len(rxnConditions)):\n",
    "        if rxnConditions[i] == conditionName:\n",
    "            newList.append(rxnFormula[i])\n",
    "            newList2.append(rxnAbbreviation[i])\n",
    "            newList3.append(conditionName)\n",
    "    return newList,newList2,newList3\n",
    "\n",
    "#Function for creating a list of GPRs given a list of Rxn names\n",
    "def getGPRsFromRxnNames(listOfRxnsNames):\n",
    "    GPR_list = []\n",
    "    for reaction in listOfRxnsNames:\n",
    "        if reaction in rxnAbbreviation:\n",
    "            GPR_list.append(rxnGPR[rxnAbbreviation.index(reaction)])\n",
    "        else:\n",
    "            GPR_list.append('')\n",
    "    return GPR_list\n",
    "     \n",
    "#Function that adds canonical reactions to overall list\n",
    "def addCanonicalRxns(listOfRxns,listOfRxnNames,listOfGPRs,listOfPathways):\n",
    "    newListRxns = listOfRxns\n",
    "    newListNames = listOfRxnNames\n",
    "    newListGPRs = listOfGPRs\n",
    "    newListPathways = listOfPathways\n",
    "    #Add  canonical reactions\n",
    "    [newListRxns, newListNames, newListPathways] = addPathway(\"Canonical\",listOfRxns,listOfRxnNames,listOfPathways)\n",
    "    #Add canonical GPRs\n",
    "    r = []\n",
    "    n = []\n",
    "    p = []\n",
    "    [r,n,p] = addPathway(\"Canonical\",r,n,p)\n",
    "    newGPRs = getGPRsFromRxnNames(n)\n",
    "    for gpr in newGPRs:\n",
    "        newListGPRs.append(gpr)\n",
    "    return newListRxns, newListNames, newListGPRs, newListPathways\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Function that generates protein-specific reactions list given a string of a Uniprot ID"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "def generateProteinSpecificRxns_A_modified(entryID):\n",
    "    #Obtain protein sequence and length\n",
    "    PSI_row = PSIM[PSIM_entries.index(str(entryID))] \n",
    "    PSI_row = PSI_row.split('\\t')\n",
    "    sequence = PSI_row[11]\n",
    "    L = float(PSI_row[2]) # Protein Length\n",
    "    MW = float(PSI_row[3]) # Molecular weight\n",
    "    PSI = []\n",
    "    Kv = 0.7\n",
    "    V = MW * 1.21 / 1000.0 # Protein Volume in nm^3\n",
    "    clathrin_coeff = int(round(29880.01 * Kv / V)) # Number of proteins per clathrin vesicle  \n",
    "    copi_coeff = int(round(143793.19 * Kv / V))\n",
    "    copii_coeff = int(round(268082.35 * Kv / V))\n",
    "    connector = ''\n",
    "    #Prepare vectors that will store reactions and components\n",
    "    protName = str(entryID)\n",
    "    rxns = []\n",
    "    rxnNames = []\n",
    "    subSystems = []\n",
    "\n",
    "    #Add translation reaction\n",
    "    translation_reaction = translate_protein(protName)\n",
    "    rxns.append(translation_reaction)\n",
    "    rxnNames.append(\"TRANSLATION_protein\")\n",
    "    subSystems.append(\"Translation\")\n",
    "    for i in [0,4,5,6,7,8,9,10]:\n",
    "        PSI.append(PSI_row[i])  #This is the vector from the PSIM that corresponds to the given protein \n",
    "\t\t\t\t\t\t\t\t\t\t  #[entry,SP,DSB,GPI,NG,OG,TMD,SubCellLoc] \n",
    "\n",
    "#------------------------------------------------------------------------------ \n",
    "    #Add translocation reactions PSI[1] = \"Signal Peptide\"\n",
    "    \n",
    "    if PSI[1] == '0': #If it doesn't have signal peptide then ignore\n",
    "        GPRs = getGPRsFromRxnNames(rxnNames)\n",
    "        #Change the reaction names and their formulas to include the UniProt Identifier\n",
    "        for i in range(len(rxns)):\n",
    "            rxns[i] = insert_prot_name_in_rxnFormula(rxns[i],protName)\n",
    "        for i in range(len(rxnNames)):\n",
    "            rxnNames[i] = insert_prot_name_in_rxnName(rxnNames[i],protName)\n",
    "        rxns.append(protName + '[c] -> ')\n",
    "        rxnNames.append(protName + '_Final_demand')\n",
    "        GPRs.append('')\n",
    "        subSystems.append('Exchange/demand reaction')\n",
    "        return rxns,rxnNames,GPRs,subSystems\n",
    "        \n",
    "    elif PSI[1] == '1': #Translocate protein if it has signal peptide\n",
    "        if L <= 160:\n",
    "            [rxns,rxnNames,subSystems] = addPathway(\"Post-translational Translocation\",rxns,rxnNames,subSystems)\n",
    "        if PSI[7] == '[e]' or PSI[7] == '':\n",
    "            [rxns,rxnNames,subSystems] = addPathway(\"Post-translational Translocation (Secretory protein)\",rxns,rxnNames,subSystems)\n",
    "        if PSI[7] == \"[pm]\" or PSI[6] != '0':\n",
    "            [rxns,rxnNames,subSystems] = addPathway(\"Post-translational Translocation (Tail anchored membrane protein)\",rxns,rxnNames,subSystems)\n",
    "        else:\n",
    "            [rxns,rxnNames,subSystems] = addPathway(\"Translocation\",rxns,rxnNames,subSystems)\n",
    "        \n",
    "        number_BiP = L/40 #Number of BiPs depends on protein length http://www.cshperspectives.com/content/5/5/a013201.full\n",
    "        for i in range(len(rxns)):\n",
    "            rxns[i] = rxns[i].replace(\"!\",str(number_BiP))\n",
    "        connector = 'XXX[r]'\n",
    "        \n",
    "#------------------------------------------------------------------------------\n",
    "    #Add Disulphide Bond Reactions\n",
    "    if PSI[2] != '0':\n",
    "        number_DSB = PSI[2]\n",
    "        DSBrxns = []\n",
    "        DSBrxnNames = []\n",
    "        DSBsubSystems = []\n",
    "        DSBrxns.append(connector + ' -> XXX_preDSB[r]')\n",
    "        DSBrxnNames.append('Start_DSB')\n",
    "        DSBsubSystems.append('DSB>0')\n",
    "        [DSBrxns, DSBrxnNames, DSBsubSystems] = addPathwayFromCondition('DSB>0',DSBrxns,DSBrxnNames,DSBsubSystems)\n",
    "        for i in range(len(DSBrxns)):\n",
    "            if '?' in DSBrxns[i]:\n",
    "                if number_DSB == '1':\n",
    "                    DSBrxns[i] = DSBrxns[i].replace('? ','')\n",
    "                else:\n",
    "                    DSBrxns[i] = DSBrxns[i].replace('?',number_DSB)\n",
    "        \n",
    "        connector = 'XXX_DSB[r]'\n",
    "\n",
    "        for reaction in DSBrxns:\n",
    "            rxns.append(reaction)\n",
    "        for reactionName in DSBrxnNames:\n",
    "            rxnNames.append(reactionName)\n",
    "        for pathwayName in DSBsubSystems:\n",
    "            subSystems.append(pathwayName)\n",
    "#------------------------------------------------------------------------------ \n",
    "    #Add GPI reactions\n",
    "    if PSI[3] == '1':\n",
    "        rxns.append(connector + ' -> XXX_preGPI[r]')\n",
    "        rxnNames.append('Start_GPI')\n",
    "        subSystems.append('GPI=1')\n",
    "        [rxns,rxnNames,subSystems] = addPathwayFromCondition('GPI=1',rxns,rxnNames,subSystems)\n",
    "        connector = 'XXX-dgpi_hs[r]'\n",
    "#------------------------------------------------------------------------------    \n",
    "    #Add N-glycosylation reactions\n",
    "    if PSI[4] != '0': #Has N-glycans?\n",
    "        rxns.append(connector + ' -> XXX_preNG[r]')\n",
    "        rxnNames.append('Start_NG')\n",
    "        subSystems.append('Golgi processing N')\n",
    "        number_Nglycans = PSI[4] #Get number of N-Glycans\n",
    "        NGlyrxns = []\n",
    "        NGlyrxnNames = []\n",
    "        NGlysubSystems = []\n",
    "        \n",
    "        [NGlyrxns,NGlyrxnNames,NGlysubSystems] = addPathwayFromCondition('NG>0',NGlyrxns,NGlyrxnNames,NGlysubSystems)\n",
    "        [NGlyrxns,NGlyrxnNames,NGlysubSystems] = addPathway('Golgi processing N',NGlyrxns,NGlyrxnNames,NGlysubSystems)\n",
    "        for i in range(len(NGlyrxns)): #Change the '?' for the number of N-glycans\n",
    "            if NGlyrxns[i] == 'XXX-M5-unfold-UBIQP[c] + ? h2o[c] + RAD23A[c] =>  XXX-unfold-UBIQP-RAD23A[c] + ? acgam[c] + ? man[c]':\n",
    "                h2o = str(7*int(number_Nglycans))\n",
    "                acgam = str(2*int(number_Nglycans))\n",
    "                man = str(5*int(number_Nglycans))\n",
    "                NGlyrxns[i] = 'XXX-M5-unfold-UBIQP[c] + ' + h2o +' h2o[c] + RAD23A[c] =>  XXX-unfold-UBIQP[c]-RAD23A[c] + ' + acgam + ' acgam[c] + ' + man +' man[c]'\n",
    "            if '?' in NGlyrxns[i]:\n",
    "                    if number_Nglycans == '1':\n",
    "                        NGlyrxns[i] = NGlyrxns[i].replace('? ', '')\n",
    "                    else:\n",
    "                        NGlyrxns[i] = NGlyrxns[i].replace('?', number_Nglycans)\n",
    "\n",
    "        for reaction in NGlyrxns:\n",
    "            rxns.append(reaction)\n",
    "        for reactionName in NGlyrxnNames:\n",
    "            rxnNames.append(reactionName)\n",
    "        for pathwayName in NGlysubSystems:\n",
    "            subSystems.append(pathwayName)\n",
    "        \n",
    "        connector = 'XXX-M8B[r]'\n",
    "           \n",
    "#------------------------------------------------------------------------------\n",
    "    #Add  COPII reactions\n",
    "    if connector == 'XXX-M8B[r]':\n",
    "        copii_rxns = []\n",
    "        copii_names = []\n",
    "        copii_systems = []\n",
    "        [copii_rxns, copii_names, copii_systems] =  addPathway('COPII_NG',copii_rxns,copii_names,copii_systems)       \n",
    "        for i in range(len(copii_rxns)):\n",
    "            copii_rxns[i] = copii_rxns[i].replace(\"!\",str(copii_coeff))\n",
    "            rxns.append(copii_rxns[i])\n",
    "            rxnNames.append(copii_names[i])\n",
    "            subSystems.append(copii_systems[i])\n",
    "        \n",
    "        #[rxns,rxnNames] = addPathway('COPII_NG',rxns,rxnNames)\n",
    "        connector = 'XXX-M3-GN2[g]'\n",
    "        \n",
    "        #-------------- ADD IF PROTEIN = EPO (HUMAN)--------------------------#\n",
    "        # IMPORTANT! -> Reactions in this pathway assume EPO has NG=3 and Og=1 (see PSIM)\n",
    "        if protName == 'P01588':\n",
    "            rxns.append('P01588[c] -> EPO_Human[c]')\n",
    "            rxnNames.append('make_EPO')\n",
    "            subSystems.append('Golgi processing (EPO specific)')\n",
    "            [rxns,rxnNames,subSystems] = addPathway('Golgi processing (EPO specific)',rxns,rxnNames,subSystems)\n",
    "            connector = 'XXX-M3-GN4-GL4-NA4-F[g]'\n",
    "            protName = 'EPO_Human'\n",
    "        \n",
    "    elif connector == 'XXX-dgpi_hs[r]':\n",
    "        copii_rxns = []\n",
    "        copii_names = []\n",
    "        copii_systems = []\n",
    "        [copii_rxns, copii_names, copii_systems] =  addPathway('COPII_GPI',copii_rxns,copii_names,copii_systems)       \n",
    "        for i in range(len(copii_rxns)):\n",
    "            copii_rxns[i] = copii_rxns[i].replace(\"!\",str(copii_coeff))\n",
    "            rxns.append(copii_rxns[i])\n",
    "            rxnNames.append(copii_names[i])\n",
    "            subSystems.append(copii_systems[i])\n",
    "        \n",
    "        #[rxns,rxnNames] = addPathway('COPII_GPI',rxns,rxnNames)\n",
    "        connector = 'XXX-dpgi_hs[g]'\n",
    "        \n",
    "    elif connector == 'XXX_DSB[r]':\n",
    "        [rxns,rxnNames,subSystems] = addPathway('COPII_DSB',rxns,rxnNames,subSystems)\n",
    "        \n",
    "        copii_rxns = []\n",
    "        copii_names = []\n",
    "        copii_systems = []\n",
    "        [copii_rxns, copii_names, copii_systems] =  addPathway('COPII-canonical',copii_rxns,copii_names,copii_systems)       \n",
    "        for i in range(len(copii_rxns)):\n",
    "            copii_rxns[i] = copii_rxns[i].replace(\"!\",str(copii_coeff))\n",
    "            rxns.append(copii_rxns[i])\n",
    "            rxnNames.append(copii_names[i])\n",
    "            subSystems.append(copii_systems[i])\n",
    "        \n",
    "        #[rxns,rxnNames] = addPathway('COPII-canonical',rxns,rxnNames)\n",
    "        connector = 'XXX[g]'\n",
    "    \n",
    "    elif connector == 'XXX[r]':\n",
    "        [rxns,rxnNames,subSystems] = addPathway('COPII-normal',rxns,rxnNames,subSystems)\n",
    "        \n",
    "        copii_rxns = []\n",
    "        copii_names = []\n",
    "        copii_systems = []\n",
    "        [copii_rxns, copii_names, copii_systems] =  addPathway('COPII-canonical',copii_rxns,copii_names,copii_systems)       \n",
    "        for i in range(len(copii_rxns)):\n",
    "            copii_rxns[i] = copii_rxns[i].replace(\"!\",str(copii_coeff))\n",
    "            rxns.append(copii_rxns[i])\n",
    "            rxnNames.append(copii_names[i])\n",
    "            subSystems.append(copii_systems[i])    \n",
    "        \n",
    "        #[rxns,rxnNames] = addPathway('COPII-canonical',rxns,rxnNames)\n",
    "        connector = 'XXX[g]'\n",
    "\n",
    "#------------------------------------------------------------------------------\n",
    "    #Add O-glycosylation reactions\n",
    "    if PSI[5] != '0': #Has O-glycans?    \n",
    "        rxns.append(connector + ' -> XXX_preOG[g]')\n",
    "        rxnNames.append('Start_OG')\n",
    "        subSystems.append('OG>0')\n",
    "        number_Oglycans = PSI[5] #Get number of O-Glycans\n",
    "        OGlyrxns = []\n",
    "        OGlyrxnNames = []\n",
    "        OGlysystems = []\n",
    "        \n",
    "        [OGlyrxns,OGlyrxnNames,OGlysystems] = addPathwayFromCondition('OG>0',OGlyrxns,OGlyrxnNames,OGlysystems)\n",
    "        for i in range(len(OGlyrxns)): #Change the '?' for the number of O-glycans\n",
    "            if '?' in OGlyrxns[i]:\n",
    "                    if number_Oglycans == '1':\n",
    "                        OGlyrxns[i] = OGlyrxns[i].replace('? ', '')\n",
    "                    else:\n",
    "                        OGlyrxns[i] = OGlyrxns[i].replace('?', number_Oglycans)\n",
    "        \n",
    "        for reaction in OGlyrxns:\n",
    "            rxns.append(reaction)\n",
    "        for reactionName in OGlyrxnNames:\n",
    "            rxnNames.append(reactionName)\n",
    "        for pathwayName in OGlysystems:\n",
    "            subSystems.append(pathwayName)\n",
    "        \n",
    "        connector = 'XXX-Core2[g]'\n",
    "        \n",
    "#------------------------------------------------------------------------------\n",
    "    #Stay in Golgi if protein is localized there\n",
    "    if PSI[7] == '[g]' or PSI[7] == '[gm]':\n",
    "        location = PSI[7]\n",
    "        #rxns.append(connector + ' -> XXX_mature' + location)\n",
    "        #rxnNames.append('Final_location_' + location)\n",
    "        if 'SP_degradation' in rxnNames:\n",
    "            SPaas = count_AAs(sequence[0:22]) #Amino acids in signal peptide assuming length is 22 on average\n",
    "            rxns[rxnNames.index('SP_degradation')] = substitute_AAs_count(rxns[rxnNames.index('SP_degradation')],SPaas)\n",
    "        if 'Ubiquitination_degradation' in rxnNames:\n",
    "            new_aas = count_AAs(sequence[22:])\n",
    "            rxns[rxnNames.index('Ubiquitination_degradation')] = rxns[rxnNames.index('Ubiquitination_degradation')].replace(\"!\", \"?\")\n",
    "            rxns[rxnNames.index('Ubiquitination_degradation')] = substitute_AAs_count(rxns[rxnNames.index('Ubiquitination_degradation')], new_aas)\n",
    "            rxns[rxnNames.index('Ubiquitination_degradation')] = rxns[rxnNames.index('Ubiquitination_degradation')].replace(\"?\",str(L-22))        \n",
    "        # Add GPRs\n",
    "        GPRs = getGPRsFromRxnNames(rxnNames)\n",
    "        #Change the reaction names and their formulas to include the UniProt Identifier\n",
    "        for i in range(len(rxns)):\n",
    "            rxns[i] = insert_prot_name_in_rxnFormula(rxns[i],protName)\n",
    "        for i in range(len(rxnNames)):\n",
    "            rxnNames[i] = insert_prot_name_in_rxnName(rxnNames[i],protName)\n",
    "        rxns.append(protName + location + ' -> ')\n",
    "        rxnNames.append(protName + '_Final_demand')\n",
    "        GPRs.append('')\n",
    "        subSystems.append('Exchange/demand reaction')\n",
    "        [rxns,rxnNames,GPRs,subSystems] = addCanonicalRxns(rxns,rxnNames,GPRs,subSystems)  \n",
    "        return rxns,rxnNames,GPRs,subSystems\n",
    "#------------------------------------------------------------------------------    \n",
    "    #Add COPI\n",
    "    elif PSI[7] == '[r]' or PSI[7] == '[rm]':\n",
    "        rxns.append(connector + ' -> XXX_preCOPI[g]')\n",
    "        rxnNames.append('Start_COPI')\n",
    "        subSystems.append('COPI')\n",
    "        \n",
    "        copi_rxns = []\n",
    "        copi_names = []\n",
    "        copi_systems = []\n",
    "        [copi_rxns, copi_names, copi_systems] =  addPathway('COPI',copi_rxns,copi_names,copi_systems)       \n",
    "        for i in range(len(copi_rxns)):\n",
    "            copi_rxns[i] = copi_rxns[i].replace(\"!\",str(copi_coeff))\n",
    "            rxns.append(copi_rxns[i])\n",
    "            rxnNames.append(copi_names[i])\n",
    "            subSystems.append(copi_systems[i])\n",
    "\n",
    "        connector = 'XXX_mature[r]'\n",
    "        location = PSI[7]\n",
    "        if location == '[r]':\n",
    "            rxns.append(connector + ' -> ')\n",
    "            rxnNames.append(protName + '_Final_demand')\n",
    "            subSystems.append('Exchange/demand reaction')\n",
    "        elif location == '[rm]':\n",
    "            rxns.append(connector + ' -> XXX_mature' + location)\n",
    "            rxnNames.append('Final_location_' + location)\n",
    "            subSystems.append('Transport, COPI vesicles')\n",
    "            rxns.append('XXX_mature' + location + ' -> ')\n",
    "            rxnNames.append(protName + '_Final_demand')\n",
    "            subSystems.append('Exchange/demand reaction')\n",
    "        \n",
    "#------------------------------------------------------------------------------\n",
    "    #Add Clathrin vesicles\n",
    "    elif PSI[7]=='[x]' or PSI[7]=='[l]' or PSI[7]=='[d]':\n",
    "        rxns.append(connector + ' -> XXX-preClathrin[g]')\n",
    "        rxnNames.append('Start_Clathrin_vesicle')\n",
    "        subSystems.append('Clathrin vesicles')\n",
    "        \n",
    "        clath_rxns = []\n",
    "        clath_names = []\n",
    "        clath_systems = []\n",
    "        [clath_rxns, clath_names, clath_systems] =  addPathway('Clathrin vesicles',clath_rxns,clath_names,clath_systems)       \n",
    "        for i in range(len(clath_rxns)):\n",
    "            clath_rxns[i] = clath_rxns[i].replace(\"!\",str(clathrin_coeff))\n",
    "            rxns.append(clath_rxns[i])\n",
    "            rxnNames.append(clath_names[i])\n",
    "            subSystems.append('Clathrin vesicles')\n",
    "        \n",
    "        connector = 'XXX_mature[cv]'\n",
    "        location = PSI[7]\n",
    "        rxns.append(connector + ' -> XXX_mature' + location)\n",
    "        rxnNames.append('Final_location_' + location)\n",
    "        subSystems.append('Transport, Clathrin vesicles')\n",
    "        rxns.append('XXX_mature' + location + ' -> ')\n",
    "        rxnNames.append(protName + '_Final_demand')\n",
    "        subSystems.append('Exchange/demand reaction')\n",
    "        \n",
    "#------------------------------------------------------------------------------\n",
    "    #Send to corresponding location\n",
    "    else:\n",
    "        location = PSI[7]\n",
    "        rxns.append(connector + ' -> XXX-preSV[g]')\n",
    "        rxnNames.append('Start_Secretion')\n",
    "        subSystems.append('SV')\n",
    "        \n",
    "        sv_rxns = []\n",
    "        sv_names = []\n",
    "        sv_systems = []\n",
    "        [sv_rxns, sv_names, sv_systems] =  addPathway('SV',sv_rxns, sv_names, sv_systems)       \n",
    "        for i in range(len(sv_rxns)):\n",
    "            sv_rxns[i] = sv_rxns[i].replace(\"!\",str(clathrin_coeff))\n",
    "            rxns.append(sv_rxns[i])\n",
    "            rxnNames.append(sv_names[i])\n",
    "            subSystems.append(sv_systems[i])      \n",
    "        \n",
    "        #[rxns,rxnNames] = addPathway(\"SV\",rxns,rxnNames)\n",
    "        if location == '':\n",
    "            location = '[e]'\n",
    "        rxns.append('XXX_mature[sv]' + ' -> XXX_mature' + location)\n",
    "        rxnNames.append('Final_location_' + location)\n",
    "        subSystems.append('Transport, extracellular')\n",
    "        rxns.append('XXX_mature' + location + ' -> ')\n",
    "        rxnNames.append('Final_demand')\n",
    "        subSystems.append('Exchange/demand reaction')\n",
    "#------------------------------------------------------------------------------\n",
    "    #Add coeffcients to SP_degradation and Ubiquitin_degradation reactions (if applicable)\n",
    "    if 'SP_degradation' in rxnNames:\n",
    "        SPaas = count_AAs(sequence[0:22]) #Amino acids in signal peptide assuming length is 22 on average\n",
    "        rxns[rxnNames.index('SP_degradation')] = substitute_AAs_count(rxns[rxnNames.index('SP_degradation')],SPaas)\n",
    "    if 'Ubiquitination_degradation' in rxnNames:\n",
    "        new_aas = count_AAs(sequence[22:])\n",
    "        rxns[rxnNames.index('Ubiquitination_degradation')] = rxns[rxnNames.index('Ubiquitination_degradation')].replace(\"!\", \"?\")\n",
    "        rxns[rxnNames.index('Ubiquitination_degradation')] = substitute_AAs_count(rxns[rxnNames.index('Ubiquitination_degradation')], new_aas)\n",
    "        rxns[rxnNames.index('Ubiquitination_degradation')] = rxns[rxnNames.index('Ubiquitination_degradation')].replace(\"?\",str(L-22))\n",
    "#------------------------------------------------------------------------------\n",
    "    GPRs = getGPRsFromRxnNames(rxnNames)\n",
    "#------------------------------------------------------------------------------\n",
    "    #Change the reaction names and their formulas to include the UniProt Identifier\n",
    "    for i in range(len(rxns)):\n",
    "        rxns[i] = insert_prot_name_in_rxnFormula(rxns[i],protName)\n",
    "    for i in range(len(rxnNames)):\n",
    "        rxnNames[i] = insert_prot_name_in_rxnName(rxnNames[i],protName)\n",
    "    [rxns,rxnNames,GPRs,subSystems] = addCanonicalRxns(rxns,rxnNames,GPRs,subSystems)  \n",
    "    return rxns,rxnNames,GPRs,subSystems"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Load recon2.2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "model = cobra.io.load_matlab_model('recon2.2.mat')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Create recon2.2s"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P60568[c]' created\n",
      "unknown metabolite 'BAT3[c]' created\n",
      "unknown metabolite 'CALM[c]' created\n",
      "unknown metabolite 'P60568-BAT3-CALM[c]' created\n",
      "unknown metabolite 'ASNA1-atp[c]' created\n",
      "unknown metabolite 'P60568-ASNA1-atp[c]' created\n",
      "unknown metabolite 'SEC61C[r]' created\n",
      "unknown metabolite 'P60568-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'SEC62[r]' created\n",
      "unknown metabolite 'SEC63[r]' created\n",
      "unknown metabolite 'TRAM1[r]' created\n",
      "unknown metabolite 'TRAP[r]' created\n",
      "unknown metabolite 'DNAJC1[r]' created\n",
      "unknown metabolite 'SERP1[r]' created\n",
      "unknown metabolite 'BiP-adp[r]' created\n",
      "unknown metabolite 'P60568-BiP-adp[r]' created\n",
      "unknown metabolite 'Translocase[r]' created\n",
      "unknown metabolite 'ASNA1-adp[c]' created\n",
      "unknown metabolite 'P60568[r]' created\n",
      "unknown metabolite 'BiP-atp[r]' created\n",
      "unknown metabolite 'SRP[c]' created\n",
      "unknown metabolite 'P60568-SRP[c]' created\n",
      "unknown metabolite 'SRPR[r]' created\n",
      "unknown metabolite 'RRBP1[r]' created\n",
      "unknown metabolite 'P60568-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P60568-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P60568-SEC61C[c]' created\n",
      "unknown metabolite 'SRP14[c]' created\n",
      "unknown metabolite 'SRP19[c]' created\n",
      "unknown metabolite 'SRP54[c]' created\n",
      "unknown metabolite 'SRP68[c]' created\n",
      "unknown metabolite 'SRP72[c]' created\n",
      "unknown metabolite 'SRPRA[r]' created\n",
      "unknown metabolite 'SRPRB[r]' created\n",
      "unknown metabolite 'SPC[r]' created\n",
      "unknown metabolite 'P60568-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P60568-sp[r]' created\n",
      "unknown metabolite 'P60568_preDSB[r]' created\n",
      "unknown metabolite 'P4HB-ox[r]' created\n",
      "unknown metabolite 'P60568-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P60568_DSB[r]' created\n",
      "unknown metabolite 'P4HB[r]' created\n",
      "unknown metabolite 'LMAN1L[r]' created\n",
      "unknown metabolite 'P60568-LMAN1L[r]' created\n",
      "unknown metabolite 'SEC23A[c]' created\n",
      "unknown metabolite 'SEC24A[c]' created\n",
      "unknown metabolite 'SEC13[c]' created\n",
      "unknown metabolite 'SEC31A[c]' created\n",
      "unknown metabolite 'BET1[c]' created\n",
      "unknown metabolite 'SEC22B[c]' created\n",
      "unknown metabolite 'SAR1B-gtp[c]' created\n",
      "unknown metabolite 'RAB1A-gtp[c]' created\n",
      "unknown metabolite 'SEC16[c]' created\n",
      "unknown metabolite 'P60568-COPII-coated[c]' created\n",
      "unknown metabolite 'P60568-COPII-uncoated[c]' created\n",
      "unknown metabolite 'SAR1B-gdp[c]' created\n",
      "unknown metabolite 'GOSR1[c]' created\n",
      "unknown metabolite 'STX5[c]' created\n",
      "unknown metabolite 'P60568[g]' created\n",
      "unknown metabolite 'RAB1A-gdp[c]' created\n",
      "unknown metabolite 'BET1-SEC22B-GOSR1-STX5[c]' created\n",
      "unknown metabolite 'P60568_preOG[g]' created\n",
      "unknown metabolite 'P60568-acgal[g]' created\n",
      "unknown metabolite 'COSMC[g]' created\n",
      "unknown metabolite 'P60568-Core1[g]' created\n",
      "unknown metabolite 'P60568-Core2[g]' created\n",
      "unknown metabolite 'P60568-preSV[g]' created\n",
      "unknown metabolite 'ARF1-gtp[c]' created\n",
      "unknown metabolite 'CLTA[c]' created\n",
      "unknown metabolite 'CLTB[c]' created\n",
      "unknown metabolite 'CLTC[c]' created\n",
      "unknown metabolite 'AP1B1[c]' created\n",
      "unknown metabolite 'AP1M1[c]' created\n",
      "unknown metabolite 'AP1S1[c]' created\n",
      "unknown metabolite 'AP1S2[c]' created\n",
      "unknown metabolite 'AP1G1[c]' created\n",
      "unknown metabolite 'AP3D1[c]' created\n",
      "unknown metabolite 'AP3M1[c]' created\n",
      "unknown metabolite 'AP3M2[c]' created\n",
      "unknown metabolite 'AP3B2[c]' created\n",
      "unknown metabolite 'AP3S2[c]' created\n",
      "unknown metabolite 'STX1B[c]' created\n",
      "unknown metabolite 'STX4[c]' created\n",
      "unknown metabolite 'RAB8B-gtp[c]' created\n",
      "unknown metabolite 'EXOCYST[c]' created\n",
      "unknown metabolite 'P60568-AP-coated[c]' created\n",
      "unknown metabolite 'STX19[c]' created\n",
      "unknown metabolite 'STX11[c]' created\n",
      "unknown metabolite 'STX16[c]' created\n",
      "unknown metabolite 'P60568_mature[sv]' created\n",
      "unknown metabolite 'ARF1-gdp[c]' created\n",
      "unknown metabolite 'STX1B-4-11-16-19[c]' created\n",
      "unknown metabolite 'RAB8B-gdp[c]' created\n",
      "unknown metabolite 'P60568_mature[e]' created\n",
      "unknown metabolite 'SEC61A1[r]' created\n",
      "unknown metabolite 'SEC61B[r]' created\n",
      "unknown metabolite 'SEC61G[r]' created\n",
      "unknown metabolite 'SSR1[r]' created\n",
      "unknown metabolite 'SSR2[r]' created\n",
      "unknown metabolite 'SSR3[r]' created\n",
      "unknown metabolite 'SSR4[r]' created\n",
      "unknown metabolite 'ARF1[c]' created\n",
      "unknown metabolite 'SAR1B[c]' created\n",
      "unknown metabolite 'RAB1A[c]' created\n",
      "unknown metabolite 'RAB6B[c]' created\n",
      "unknown metabolite 'RAB6B-gdp[c]' created\n",
      "unknown metabolite 'RAB8B[c]' created\n",
      "unknown metabolite 'RAB6B-gtp[c]' created\n",
      "unknown metabolite 'BiP[r]' created\n",
      "unknown metabolite 'STX6-7-12[c]' created\n",
      "unknown metabolite 'NSF[c]' created\n",
      "unknown metabolite 'NAPA[c]' created\n",
      "unknown metabolite 'NSF(6)NAPA[c]' created\n",
      "unknown metabolite 'STX6[c]' created\n",
      "unknown metabolite 'STX7[c]' created\n",
      "unknown metabolite 'STX12[c]' created\n",
      "unknown metabolite 'BET1-SEC22B-USE1-STX8-STX18[c]' created\n",
      "unknown metabolite 'USE1[c]' created\n",
      "unknown metabolite 'STX8[c]' created\n",
      "unknown metabolite 'STX18[c]' created\n",
      "unknown metabolite 'NPLOC4[c]' created\n",
      "unknown metabolite 'UFD1L[c]' created\n",
      "unknown metabolite 'VCP[c]' created\n",
      "unknown metabolite 'DERL1[r]' created\n",
      "unknown metabolite 'VIMP[c]' created\n",
      "unknown metabolite 'DERL3[r]' created\n",
      "unknown metabolite 'BCAP31[r]' created\n",
      "unknown metabolite 'retroTranslocase[c]' created\n",
      "unknown metabolite 'PROTA[c]' created\n",
      "unknown metabolite 'PROTB[c]' created\n",
      "unknown metabolite 'PROASCHAP1[c]' created\n",
      "unknown metabolite 'PROASCHAP2[c]' created\n",
      "unknown metabolite 'PROASCHAP3[c]' created\n",
      "unknown metabolite 'PROASCHAP4[c]' created\n",
      "unknown metabolite 'PROTREG4[c]' created\n",
      "unknown metabolite 'PROTREG5[c]' created\n",
      "unknown metabolite 'PROTREG6[c]' created\n",
      "unknown metabolite 'PROTREG7[c]' created\n",
      "unknown metabolite 'PROTREG8[c]' created\n",
      "unknown metabolite 'PROTREG9[c]' created\n",
      "unknown metabolite 'PROTREG10[c]' created\n",
      "unknown metabolite 'PROTREG11[c]' created\n",
      "unknown metabolite 'PROTREG12[c]' created\n",
      "unknown metabolite 'PROTREG13[c]' created\n",
      "unknown metabolite 'PROTREG14[c]' created\n",
      "unknown metabolite 'Proteasome[c]' created\n",
      "unknown metabolite 'ASNA1[c]' created\n",
      "unknown metabolite 'CALR[r]' created\n",
      "unknown metabolite 'ca2[r]' created\n",
      "unknown metabolite 'CALR-ca2[r]' created\n",
      "unknown metabolite 'gthox[r]' created\n",
      "unknown metabolite 'ERO1LB-ox[r]' created\n",
      "unknown metabolite 'ERO1LB[r]' created\n",
      "unknown metabolite 'ERO1A[r]' created\n",
      "unknown metabolite 'ERO1A-ox[r]' created\n",
      "unknown metabolite 'EXOC1[c]' created\n",
      "unknown metabolite 'EXOC2[c]' created\n",
      "unknown metabolite 'EXOC3[c]' created\n",
      "unknown metabolite 'EXOC4[c]' created\n",
      "unknown metabolite 'EXOC5[c]' created\n",
      "unknown metabolite 'EXOC6B[c]' created\n",
      "unknown metabolite 'EXOC7[c]' created\n",
      "unknown metabolite 'EXOC8[c]' created\n",
      "unknown metabolite 'SPCS1[r]' created\n",
      "unknown metabolite 'SPCS2[r]' created\n",
      "unknown metabolite 'SPCS3[r]' created\n",
      "unknown metabolite 'SEC11A[r]' created\n",
      "unknown metabolite 'SEC11C[r]' created\n",
      "unknown metabolite 'DDOST[r]' created\n",
      "unknown metabolite 'RPN1[r]' created\n",
      "unknown metabolite 'RPN2[r]' created\n",
      "unknown metabolite 'DAD1[r]' created\n",
      "unknown metabolite 'STT3A[r]' created\n",
      "unknown metabolite 'STT3B[r]' created\n",
      "unknown metabolite 'OST[r]' created\n",
      "unknown metabolite 'g3m8mpdol[r]' created\n",
      "unknown metabolite 'doldp_L[r]' created\n",
      "unknown metabolite 'doldp[r]' created\n",
      "unknown metabolite 'PSMA1[c]' created\n",
      "unknown metabolite 'PSMA2[c]' created\n",
      "unknown metabolite 'PSMA3[c]' created\n",
      "unknown metabolite 'PSMA4[c]' created\n",
      "unknown metabolite 'PSMA5[c]' created\n",
      "unknown metabolite 'PSMA7[c]' created\n",
      "unknown metabolite 'PSMA8[c]' created\n",
      "unknown metabolite 'PSMB1[c]' created\n",
      "unknown metabolite 'PSMB2[c]' created\n",
      "unknown metabolite 'PSMB3[c]' created\n",
      "unknown metabolite 'PSMB4[c]' created\n",
      "unknown metabolite 'PSMB5[c]' created\n",
      "unknown metabolite 'PSMB6[c]' created\n",
      "unknown metabolite 'PSMB7[c]' created\n",
      "unknown metabolite 'PROTB9[c]' created\n",
      "unknown metabolite 'PROTB10[c]' created\n",
      "unknown metabolite 'PROTB11[c]' created\n",
      "unknown metabolite 'P01589[c]' created\n",
      "unknown metabolite 'P01589-ASNA1-atp[c]' created\n",
      "unknown metabolite 'WRB[r]' created\n",
      "unknown metabolite 'P01589-ASNA1-atp-WRB[r]' created\n",
      "unknown metabolite 'P01589[r]' created\n",
      "unknown metabolite 'P01589_preDSB[r]' created\n",
      "unknown metabolite 'P01589-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P01589_DSB[r]' created\n",
      "unknown metabolite 'P01589_preNG[r]' created\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P01589-G3M9[r]' created\n",
      "unknown metabolite 'P01589-M9-misfold[r]' created\n",
      "unknown metabolite 'P01589-M8C-misfold[r]' created\n",
      "unknown metabolite 'P01589-M7A-misfold[r]' created\n",
      "unknown metabolite 'P01589-M6-misfold[r]' created\n",
      "unknown metabolite 'P01589-M5-misfold[r]' created\n",
      "unknown metabolite 'OS9[r]' created\n",
      "unknown metabolite 'HSP90B1[r]' created\n",
      "unknown metabolite 'P01589-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P01589-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'SYVN1[r]' created\n",
      "unknown metabolite 'P01589-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'UBIQP[c]' created\n",
      "unknown metabolite 'P01589-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P01589-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'SEL1[r]' created\n",
      "unknown metabolite 'RAD23A[c]' created\n",
      "unknown metabolite 'P01589-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P01589-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P01589-G2M9[r]' created\n",
      "unknown metabolite 'P01589-preG1M9[r]' created\n",
      "unknown metabolite 'CANX[r]' created\n",
      "unknown metabolite 'P01589-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P01589-G1M9[r]' created\n",
      "unknown metabolite 'P01589-M9[r]' created\n",
      "unknown metabolite 'UGGT1[r]' created\n",
      "unknown metabolite 'P01589-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P01589-M8B[r]' created\n",
      "unknown metabolite 'P01589-M8B[g]' created\n",
      "unknown metabolite 'P01589-M7[g]' created\n",
      "unknown metabolite 'P01589-M6[g]' created\n",
      "unknown metabolite 'P01589-M5[g]' created\n",
      "unknown metabolite 'P01589-M5-GN[g]' created\n",
      "unknown metabolite 'P01589-M4-GN[g]' created\n",
      "unknown metabolite 'P01589-M3-GN[g]' created\n",
      "unknown metabolite 'P01589-M3-GN2[g]' created\n",
      "unknown metabolite 'P01589-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P01589-COPII-coated[c]' created\n",
      "unknown metabolite 'P01589-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P01589_preOG[g]' created\n",
      "unknown metabolite 'P01589-acgal[g]' created\n",
      "unknown metabolite 'P01589-Core1[g]' created\n",
      "unknown metabolite 'P01589-Core2[g]' created\n",
      "unknown metabolite 'P01589-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P01589-AP-coated[c]' created\n",
      "unknown metabolite 'P01589_mature[sv]' created\n",
      "unknown metabolite 'P01589_mature[pm]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P14784[c]' created\n",
      "unknown metabolite 'P14784-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P14784-ASNA1-atp-WRB[r]' created\n",
      "unknown metabolite 'P14784[r]' created\n",
      "unknown metabolite 'P14784_preDSB[r]' created\n",
      "unknown metabolite 'P14784-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P14784_DSB[r]' created\n",
      "unknown metabolite 'P14784_preNG[r]' created\n",
      "unknown metabolite 'P14784-G3M9[r]' created\n",
      "unknown metabolite 'P14784-M9-misfold[r]' created\n",
      "unknown metabolite 'P14784-M8C-misfold[r]' created\n",
      "unknown metabolite 'P14784-M7A-misfold[r]' created\n",
      "unknown metabolite 'P14784-M6-misfold[r]' created\n",
      "unknown metabolite 'P14784-M5-misfold[r]' created\n",
      "unknown metabolite 'P14784-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P14784-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P14784-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P14784-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P14784-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P14784-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P14784-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P14784-G2M9[r]' created\n",
      "unknown metabolite 'P14784-preG1M9[r]' created\n",
      "unknown metabolite 'P14784-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P14784-G1M9[r]' created\n",
      "unknown metabolite 'P14784-M9[r]' created\n",
      "unknown metabolite 'P14784-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P14784-M8B[r]' created\n",
      "unknown metabolite 'P14784-M8B[g]' created\n",
      "unknown metabolite 'P14784-M7[g]' created\n",
      "unknown metabolite 'P14784-M6[g]' created\n",
      "unknown metabolite 'P14784-M5[g]' created\n",
      "unknown metabolite 'P14784-M5-GN[g]' created\n",
      "unknown metabolite 'P14784-M4-GN[g]' created\n",
      "unknown metabolite 'P14784-M3-GN[g]' created\n",
      "unknown metabolite 'P14784-M3-GN2[g]' created\n",
      "unknown metabolite 'P14784-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P14784-COPII-coated[c]' created\n",
      "unknown metabolite 'P14784-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P14784-preSV[g]' created\n",
      "unknown metabolite 'P14784-AP-coated[c]' created\n",
      "unknown metabolite 'P14784_mature[sv]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P14784_mature[pm]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P31785[c]' created\n",
      "unknown metabolite 'P31785-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P31785-ASNA1-atp-WRB[r]' created\n",
      "unknown metabolite 'P31785[r]' created\n",
      "unknown metabolite 'P31785_preDSB[r]' created\n",
      "unknown metabolite 'P31785-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P31785_DSB[r]' created\n",
      "unknown metabolite 'P31785_preNG[r]' created\n",
      "unknown metabolite 'P31785-G3M9[r]' created\n",
      "unknown metabolite 'P31785-M9-misfold[r]' created\n",
      "unknown metabolite 'P31785-M8C-misfold[r]' created\n",
      "unknown metabolite 'P31785-M7A-misfold[r]' created\n",
      "unknown metabolite 'P31785-M6-misfold[r]' created\n",
      "unknown metabolite 'P31785-M5-misfold[r]' created\n",
      "unknown metabolite 'P31785-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P31785-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P31785-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P31785-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P31785-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P31785-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P31785-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P31785-G2M9[r]' created\n",
      "unknown metabolite 'P31785-preG1M9[r]' created\n",
      "unknown metabolite 'P31785-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P31785-G1M9[r]' created\n",
      "unknown metabolite 'P31785-M9[r]' created\n",
      "unknown metabolite 'P31785-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P31785-M8B[r]' created\n",
      "unknown metabolite 'P31785-M8B[g]' created\n",
      "unknown metabolite 'P31785-M7[g]' created\n",
      "unknown metabolite 'P31785-M6[g]' created\n",
      "unknown metabolite 'P31785-M5[g]' created\n",
      "unknown metabolite 'P31785-M5-GN[g]' created\n",
      "unknown metabolite 'P31785-M4-GN[g]' created\n",
      "unknown metabolite 'P31785-M3-GN[g]' created\n",
      "unknown metabolite 'P31785-M3-GN2[g]' created\n",
      "unknown metabolite 'P31785-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P31785-COPII-coated[c]' created\n",
      "unknown metabolite 'P31785-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P31785-preSV[g]' created\n",
      "unknown metabolite 'P31785-AP-coated[c]' created\n",
      "unknown metabolite 'P31785_mature[sv]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P31785_mature[pm]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P05231[c]' created\n",
      "unknown metabolite 'P05231-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P05231-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P05231-BiP-adp[r]' created\n",
      "unknown metabolite 'P05231[r]' created\n",
      "unknown metabolite 'P05231-SRP[c]' created\n",
      "unknown metabolite 'P05231-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P05231-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P05231-SEC61C[c]' created\n",
      "unknown metabolite 'P05231-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P05231-sp[r]' created\n",
      "unknown metabolite 'P05231_preDSB[r]' created\n",
      "unknown metabolite 'P05231-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P05231_DSB[r]' created\n",
      "unknown metabolite 'P05231_preNG[r]' created\n",
      "unknown metabolite 'P05231-G3M9[r]' created\n",
      "unknown metabolite 'P05231-M9-misfold[r]' created\n",
      "unknown metabolite 'P05231-M8C-misfold[r]' created\n",
      "unknown metabolite 'P05231-M7A-misfold[r]' created\n",
      "unknown metabolite 'P05231-M6-misfold[r]' created\n",
      "unknown metabolite 'P05231-M5-misfold[r]' created\n",
      "unknown metabolite 'P05231-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P05231-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P05231-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P05231-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P05231-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P05231-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P05231-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P05231-G2M9[r]' created\n",
      "unknown metabolite 'P05231-preG1M9[r]' created\n",
      "unknown metabolite 'P05231-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P05231-G1M9[r]' created\n",
      "unknown metabolite 'P05231-M9[r]' created\n",
      "unknown metabolite 'P05231-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P05231-M8B[r]' created\n",
      "unknown metabolite 'P05231-M8B[g]' created\n",
      "unknown metabolite 'P05231-M7[g]' created\n",
      "unknown metabolite 'P05231-M6[g]' created\n",
      "unknown metabolite 'P05231-M5[g]' created\n",
      "unknown metabolite 'P05231-M5-GN[g]' created\n",
      "unknown metabolite 'P05231-M4-GN[g]' created\n",
      "unknown metabolite 'P05231-M3-GN[g]' created\n",
      "unknown metabolite 'P05231-M3-GN2[g]' created\n",
      "unknown metabolite 'P05231-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P05231-COPII-coated[c]' created\n",
      "unknown metabolite 'P05231-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P05231-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P05231-AP-coated[c]' created\n",
      "unknown metabolite 'P05231_mature[sv]' created\n",
      "unknown metabolite 'P05231_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P08887[c]' created\n",
      "unknown metabolite 'P08887-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P08887-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P08887-BiP-adp[r]' created\n",
      "unknown metabolite 'P08887[r]' created\n",
      "unknown metabolite 'P08887-ASNA1-atp-WRB[r]' created\n",
      "unknown metabolite 'P08887_preDSB[r]' created\n",
      "unknown metabolite 'P08887-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P08887_DSB[r]' created\n",
      "unknown metabolite 'P08887_preNG[r]' created\n",
      "unknown metabolite 'P08887-G3M9[r]' created\n",
      "unknown metabolite 'P08887-M9-misfold[r]' created\n",
      "unknown metabolite 'P08887-M8C-misfold[r]' created\n",
      "unknown metabolite 'P08887-M7A-misfold[r]' created\n",
      "unknown metabolite 'P08887-M6-misfold[r]' created\n",
      "unknown metabolite 'P08887-M5-misfold[r]' created\n",
      "unknown metabolite 'P08887-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P08887-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P08887-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P08887-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P08887-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P08887-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P08887-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P08887-G2M9[r]' created\n",
      "unknown metabolite 'P08887-preG1M9[r]' created\n",
      "unknown metabolite 'P08887-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P08887-G1M9[r]' created\n",
      "unknown metabolite 'P08887-M9[r]' created\n",
      "unknown metabolite 'P08887-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P08887-M8B[r]' created\n",
      "unknown metabolite 'P08887-M8B[g]' created\n",
      "unknown metabolite 'P08887-M7[g]' created\n",
      "unknown metabolite 'P08887-M6[g]' created\n",
      "unknown metabolite 'P08887-M5[g]' created\n",
      "unknown metabolite 'P08887-M5-GN[g]' created\n",
      "unknown metabolite 'P08887-M4-GN[g]' created\n",
      "unknown metabolite 'P08887-M3-GN[g]' created\n",
      "unknown metabolite 'P08887-M3-GN2[g]' created\n",
      "unknown metabolite 'P08887-M8B-LMAN1L[r]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P08887-COPII-coated[c]' created\n",
      "unknown metabolite 'P08887-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P08887-preSV[g]' created\n",
      "unknown metabolite 'P08887-AP-coated[c]' created\n",
      "unknown metabolite 'P08887_mature[sv]' created\n",
      "unknown metabolite 'P08887_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P40189[c]' created\n",
      "unknown metabolite 'P40189-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P40189-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P40189-BiP-adp[r]' created\n",
      "unknown metabolite 'P40189[r]' created\n",
      "unknown metabolite 'P40189-ASNA1-atp-WRB[r]' created\n",
      "unknown metabolite 'P40189_preDSB[r]' created\n",
      "unknown metabolite 'P40189-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P40189_DSB[r]' created\n",
      "unknown metabolite 'P40189_preNG[r]' created\n",
      "unknown metabolite 'P40189-G3M9[r]' created\n",
      "unknown metabolite 'P40189-M9-misfold[r]' created\n",
      "unknown metabolite 'P40189-M8C-misfold[r]' created\n",
      "unknown metabolite 'P40189-M7A-misfold[r]' created\n",
      "unknown metabolite 'P40189-M6-misfold[r]' created\n",
      "unknown metabolite 'P40189-M5-misfold[r]' created\n",
      "unknown metabolite 'P40189-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P40189-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P40189-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P40189-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P40189-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P40189-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P40189-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P40189-G2M9[r]' created\n",
      "unknown metabolite 'P40189-preG1M9[r]' created\n",
      "unknown metabolite 'P40189-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P40189-G1M9[r]' created\n",
      "unknown metabolite 'P40189-M9[r]' created\n",
      "unknown metabolite 'P40189-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P40189-M8B[r]' created\n",
      "unknown metabolite 'P40189-M8B[g]' created\n",
      "unknown metabolite 'P40189-M7[g]' created\n",
      "unknown metabolite 'P40189-M6[g]' created\n",
      "unknown metabolite 'P40189-M5[g]' created\n",
      "unknown metabolite 'P40189-M5-GN[g]' created\n",
      "unknown metabolite 'P40189-M4-GN[g]' created\n",
      "unknown metabolite 'P40189-M3-GN[g]' created\n",
      "unknown metabolite 'P40189-M3-GN2[g]' created\n",
      "unknown metabolite 'P40189-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P40189-COPII-coated[c]' created\n",
      "unknown metabolite 'P40189-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P40189-preSV[g]' created\n",
      "unknown metabolite 'P40189-AP-coated[c]' created\n",
      "unknown metabolite 'P40189_mature[sv]' created\n",
      "unknown metabolite 'P40189_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P13232[c]' created\n",
      "unknown metabolite 'P13232-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P13232-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P13232-BiP-adp[r]' created\n",
      "unknown metabolite 'P13232[r]' created\n",
      "unknown metabolite 'P13232-SRP[c]' created\n",
      "unknown metabolite 'P13232-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P13232-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P13232-SEC61C[c]' created\n",
      "unknown metabolite 'P13232-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P13232-sp[r]' created\n",
      "unknown metabolite 'P13232_preDSB[r]' created\n",
      "unknown metabolite 'P13232-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P13232_DSB[r]' created\n",
      "unknown metabolite 'P13232_preNG[r]' created\n",
      "unknown metabolite 'P13232-G3M9[r]' created\n",
      "unknown metabolite 'P13232-M9-misfold[r]' created\n",
      "unknown metabolite 'P13232-M8C-misfold[r]' created\n",
      "unknown metabolite 'P13232-M7A-misfold[r]' created\n",
      "unknown metabolite 'P13232-M6-misfold[r]' created\n",
      "unknown metabolite 'P13232-M5-misfold[r]' created\n",
      "unknown metabolite 'P13232-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P13232-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P13232-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P13232-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P13232-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P13232-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P13232-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P13232-G2M9[r]' created\n",
      "unknown metabolite 'P13232-preG1M9[r]' created\n",
      "unknown metabolite 'P13232-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P13232-G1M9[r]' created\n",
      "unknown metabolite 'P13232-M9[r]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P13232-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P13232-M8B[r]' created\n",
      "unknown metabolite 'P13232-M8B[g]' created\n",
      "unknown metabolite 'P13232-M7[g]' created\n",
      "unknown metabolite 'P13232-M6[g]' created\n",
      "unknown metabolite 'P13232-M5[g]' created\n",
      "unknown metabolite 'P13232-M5-GN[g]' created\n",
      "unknown metabolite 'P13232-M4-GN[g]' created\n",
      "unknown metabolite 'P13232-M3-GN[g]' created\n",
      "unknown metabolite 'P13232-M3-GN2[g]' created\n",
      "unknown metabolite 'P13232-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P13232-COPII-coated[c]' created\n",
      "unknown metabolite 'P13232-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P13232-preSV[g]' created\n",
      "unknown metabolite 'P13232-AP-coated[c]' created\n",
      "unknown metabolite 'P13232_mature[sv]' created\n",
      "unknown metabolite 'P13232_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P16871[c]' created\n",
      "unknown metabolite 'P16871-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P16871-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P16871-BiP-adp[r]' created\n",
      "unknown metabolite 'P16871[r]' created\n",
      "unknown metabolite 'P16871-ASNA1-atp-WRB[r]' created\n",
      "unknown metabolite 'P16871_preDSB[r]' created\n",
      "unknown metabolite 'P16871-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P16871_DSB[r]' created\n",
      "unknown metabolite 'P16871_preNG[r]' created\n",
      "unknown metabolite 'P16871-G3M9[r]' created\n",
      "unknown metabolite 'P16871-M9-misfold[r]' created\n",
      "unknown metabolite 'P16871-M8C-misfold[r]' created\n",
      "unknown metabolite 'P16871-M7A-misfold[r]' created\n",
      "unknown metabolite 'P16871-M6-misfold[r]' created\n",
      "unknown metabolite 'P16871-M5-misfold[r]' created\n",
      "unknown metabolite 'P16871-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P16871-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P16871-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P16871-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P16871-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P16871-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P16871-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P16871-G2M9[r]' created\n",
      "unknown metabolite 'P16871-preG1M9[r]' created\n",
      "unknown metabolite 'P16871-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P16871-G1M9[r]' created\n",
      "unknown metabolite 'P16871-M9[r]' created\n",
      "unknown metabolite 'P16871-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P16871-M8B[r]' created\n",
      "unknown metabolite 'P16871-M8B[g]' created\n",
      "unknown metabolite 'P16871-M7[g]' created\n",
      "unknown metabolite 'P16871-M6[g]' created\n",
      "unknown metabolite 'P16871-M5[g]' created\n",
      "unknown metabolite 'P16871-M5-GN[g]' created\n",
      "unknown metabolite 'P16871-M4-GN[g]' created\n",
      "unknown metabolite 'P16871-M3-GN[g]' created\n",
      "unknown metabolite 'P16871-M3-GN2[g]' created\n",
      "unknown metabolite 'P16871-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P16871-COPII-coated[c]' created\n",
      "unknown metabolite 'P16871-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P16871-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P16871-AP-coated[c]' created\n",
      "unknown metabolite 'P16871_mature[sv]' created\n",
      "unknown metabolite 'P16871_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P01579[c]' created\n",
      "unknown metabolite 'P01579-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P01579-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P01579-BiP-adp[r]' created\n",
      "unknown metabolite 'P01579[r]' created\n",
      "unknown metabolite 'P01579-SRP[c]' created\n",
      "unknown metabolite 'P01579-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P01579-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P01579-SEC61C[c]' created\n",
      "unknown metabolite 'P01579-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P01579-sp[r]' created\n",
      "unknown metabolite 'P01579_preNG[r]' created\n",
      "unknown metabolite 'P01579-G3M9[r]' created\n",
      "unknown metabolite 'P01579-M9-misfold[r]' created\n",
      "unknown metabolite 'P01579-M8C-misfold[r]' created\n",
      "unknown metabolite 'P01579-M7A-misfold[r]' created\n",
      "unknown metabolite 'P01579-M6-misfold[r]' created\n",
      "unknown metabolite 'P01579-M5-misfold[r]' created\n",
      "unknown metabolite 'P01579-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P01579-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P01579-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P01579-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P01579-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P01579-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P01579-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P01579-G2M9[r]' created\n",
      "unknown metabolite 'P01579-preG1M9[r]' created\n",
      "unknown metabolite 'P01579-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P01579-G1M9[r]' created\n",
      "unknown metabolite 'P01579-M9[r]' created\n",
      "unknown metabolite 'P01579-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P01579-M8B[r]' created\n",
      "unknown metabolite 'P01579-M8B[g]' created\n",
      "unknown metabolite 'P01579-M7[g]' created\n",
      "unknown metabolite 'P01579-M6[g]' created\n",
      "unknown metabolite 'P01579-M5[g]' created\n",
      "unknown metabolite 'P01579-M5-GN[g]' created\n",
      "unknown metabolite 'P01579-M4-GN[g]' created\n",
      "unknown metabolite 'P01579-M3-GN[g]' created\n",
      "unknown metabolite 'P01579-M3-GN2[g]' created\n",
      "unknown metabolite 'P01579-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P01579-COPII-coated[c]' created\n",
      "unknown metabolite 'P01579-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P01579-preSV[g]' created\n",
      "unknown metabolite 'P01579-AP-coated[c]' created\n",
      "unknown metabolite 'P01579_mature[sv]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P01579_mature[e]' created\n",
      "unknown metabolite 'P01584[c]' created\n",
      "unknown metabolite 'P18510[c]' created\n",
      "unknown metabolite 'P18510-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P18510-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P18510-BiP-adp[r]' created\n",
      "unknown metabolite 'P18510[r]' created\n",
      "unknown metabolite 'P18510-SRP[c]' created\n",
      "unknown metabolite 'P18510-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P18510-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P18510-SEC61C[c]' created\n",
      "unknown metabolite 'P18510-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P18510-sp[r]' created\n",
      "unknown metabolite 'P18510_preDSB[r]' created\n",
      "unknown metabolite 'P18510-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P18510_DSB[r]' created\n",
      "unknown metabolite 'P18510_preNG[r]' created\n",
      "unknown metabolite 'P18510-G3M9[r]' created\n",
      "unknown metabolite 'P18510-M9-misfold[r]' created\n",
      "unknown metabolite 'P18510-M8C-misfold[r]' created\n",
      "unknown metabolite 'P18510-M7A-misfold[r]' created\n",
      "unknown metabolite 'P18510-M6-misfold[r]' created\n",
      "unknown metabolite 'P18510-M5-misfold[r]' created\n",
      "unknown metabolite 'P18510-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P18510-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P18510-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P18510-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P18510-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P18510-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P18510-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P18510-G2M9[r]' created\n",
      "unknown metabolite 'P18510-preG1M9[r]' created\n",
      "unknown metabolite 'P18510-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P18510-G1M9[r]' created\n",
      "unknown metabolite 'P18510-M9[r]' created\n",
      "unknown metabolite 'P18510-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P18510-M8B[r]' created\n",
      "unknown metabolite 'P18510-M8B[g]' created\n",
      "unknown metabolite 'P18510-M7[g]' created\n",
      "unknown metabolite 'P18510-M6[g]' created\n",
      "unknown metabolite 'P18510-M5[g]' created\n",
      "unknown metabolite 'P18510-M5-GN[g]' created\n",
      "unknown metabolite 'P18510-M4-GN[g]' created\n",
      "unknown metabolite 'P18510-M3-GN[g]' created\n",
      "unknown metabolite 'P18510-M3-GN2[g]' created\n",
      "unknown metabolite 'P18510-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P18510-COPII-coated[c]' created\n",
      "unknown metabolite 'P18510-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P18510-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P18510-AP-coated[c]' created\n",
      "unknown metabolite 'P18510_mature[sv]' created\n",
      "unknown metabolite 'P18510_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P05112[c]' created\n",
      "unknown metabolite 'P05112-BAT3-CALM[c]' created\n",
      "unknown metabolite 'P05112-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P05112-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P05112-BiP-adp[r]' created\n",
      "unknown metabolite 'P05112[r]' created\n",
      "unknown metabolite 'P05112-SRP[c]' created\n",
      "unknown metabolite 'P05112-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P05112-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P05112-SEC61C[c]' created\n",
      "unknown metabolite 'P05112-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P05112-sp[r]' created\n",
      "unknown metabolite 'P05112_preDSB[r]' created\n",
      "unknown metabolite 'P05112-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P05112_DSB[r]' created\n",
      "unknown metabolite 'P05112_preNG[r]' created\n",
      "unknown metabolite 'P05112-G3M9[r]' created\n",
      "unknown metabolite 'P05112-M9-misfold[r]' created\n",
      "unknown metabolite 'P05112-M8C-misfold[r]' created\n",
      "unknown metabolite 'P05112-M7A-misfold[r]' created\n",
      "unknown metabolite 'P05112-M6-misfold[r]' created\n",
      "unknown metabolite 'P05112-M5-misfold[r]' created\n",
      "unknown metabolite 'P05112-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P05112-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P05112-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P05112-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P05112-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P05112-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P05112-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P05112-G2M9[r]' created\n",
      "unknown metabolite 'P05112-preG1M9[r]' created\n",
      "unknown metabolite 'P05112-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P05112-G1M9[r]' created\n",
      "unknown metabolite 'P05112-M9[r]' created\n",
      "unknown metabolite 'P05112-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P05112-M8B[r]' created\n",
      "unknown metabolite 'P05112-M8B[g]' created\n",
      "unknown metabolite 'P05112-M7[g]' created\n",
      "unknown metabolite 'P05112-M6[g]' created\n",
      "unknown metabolite 'P05112-M5[g]' created\n",
      "unknown metabolite 'P05112-M5-GN[g]' created\n",
      "unknown metabolite 'P05112-M4-GN[g]' created\n",
      "unknown metabolite 'P05112-M3-GN[g]' created\n",
      "unknown metabolite 'P05112-M3-GN2[g]' created\n",
      "unknown metabolite 'P05112-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P05112-COPII-coated[c]' created\n",
      "unknown metabolite 'P05112-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P05112-preSV[g]' created"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "unknown metabolite 'P05112-AP-coated[c]' created\n",
      "unknown metabolite 'P05112_mature[sv]' created\n",
      "unknown metabolite 'P05112_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P05113[c]' created\n",
      "unknown metabolite 'P05113-BAT3-CALM[c]' created\n",
      "unknown metabolite 'P05113-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P05113-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P05113-BiP-adp[r]' created\n",
      "unknown metabolite 'P05113[r]' created\n",
      "unknown metabolite 'P05113-SRP[c]' created\n",
      "unknown metabolite 'P05113-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P05113-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P05113-SEC61C[c]' created\n",
      "unknown metabolite 'P05113-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P05113-sp[r]' created\n",
      "unknown metabolite 'P05113_preDSB[r]' created\n",
      "unknown metabolite 'P05113-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P05113_DSB[r]' created\n",
      "unknown metabolite 'P05113_preNG[r]' created\n",
      "unknown metabolite 'P05113-G3M9[r]' created\n",
      "unknown metabolite 'P05113-M9-misfold[r]' created\n",
      "unknown metabolite 'P05113-M8C-misfold[r]' created\n",
      "unknown metabolite 'P05113-M7A-misfold[r]' created\n",
      "unknown metabolite 'P05113-M6-misfold[r]' created\n",
      "unknown metabolite 'P05113-M5-misfold[r]' created\n",
      "unknown metabolite 'P05113-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P05113-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P05113-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P05113-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P05113-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P05113-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P05113-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P05113-G2M9[r]' created\n",
      "unknown metabolite 'P05113-preG1M9[r]' created\n",
      "unknown metabolite 'P05113-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P05113-G1M9[r]' created\n",
      "unknown metabolite 'P05113-M9[r]' created\n",
      "unknown metabolite 'P05113-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P05113-M8B[r]' created\n",
      "unknown metabolite 'P05113-M8B[g]' created\n",
      "unknown metabolite 'P05113-M7[g]' created\n",
      "unknown metabolite 'P05113-M6[g]' created\n",
      "unknown metabolite 'P05113-M5[g]' created\n",
      "unknown metabolite 'P05113-M5-GN[g]' created\n",
      "unknown metabolite 'P05113-M4-GN[g]' created\n",
      "unknown metabolite 'P05113-M3-GN[g]' created\n",
      "unknown metabolite 'P05113-M3-GN2[g]' created\n",
      "unknown metabolite 'P05113-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P05113-COPII-coated[c]' created\n",
      "unknown metabolite 'P05113-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P05113_preOG[g]' created\n",
      "unknown metabolite 'P05113-acgal[g]' created\n",
      "unknown metabolite 'P05113-Core1[g]' created\n",
      "unknown metabolite 'P05113-Core2[g]' created\n",
      "unknown metabolite 'P05113-preSV[g]' created\n",
      "unknown metabolite 'P05113-AP-coated[c]' created\n",
      "unknown metabolite 'P05113_mature[sv]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P05113_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P10145[c]' created\n",
      "unknown metabolite 'P10145-BAT3-CALM[c]' created\n",
      "unknown metabolite 'P10145-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P10145-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P10145-BiP-adp[r]' created\n",
      "unknown metabolite 'P10145[r]' created\n",
      "unknown metabolite 'P10145-SRP[c]' created\n",
      "unknown metabolite 'P10145-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P10145-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P10145-SEC61C[c]' created\n",
      "unknown metabolite 'P10145-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P10145-sp[r]' created\n",
      "unknown metabolite 'P10145_preDSB[r]' created\n",
      "unknown metabolite 'P10145-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P10145_DSB[r]' created\n",
      "unknown metabolite 'P10145-LMAN1L[r]' created\n",
      "unknown metabolite 'P10145-COPII-coated[c]' created\n",
      "unknown metabolite 'P10145-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P10145[g]' created\n",
      "unknown metabolite 'P10145-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P10145-AP-coated[c]' created\n",
      "unknown metabolite 'P10145_mature[sv]' created\n",
      "unknown metabolite 'P10145_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P15248[c]' created\n",
      "unknown metabolite 'P15248-BAT3-CALM[c]' created\n",
      "unknown metabolite 'P15248-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P15248-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P15248-BiP-adp[r]' created\n",
      "unknown metabolite 'P15248[r]' created\n",
      "unknown metabolite 'P15248-SRP[c]' created\n",
      "unknown metabolite 'P15248-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P15248-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P15248-SEC61C[c]' created\n",
      "unknown metabolite 'P15248-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P15248-sp[r]' created\n",
      "unknown metabolite 'P15248_preNG[r]' created\n",
      "unknown metabolite 'P15248-G3M9[r]' created\n",
      "unknown metabolite 'P15248-M9-misfold[r]' created\n",
      "unknown metabolite 'P15248-M8C-misfold[r]' created\n",
      "unknown metabolite 'P15248-M7A-misfold[r]' created\n",
      "unknown metabolite 'P15248-M6-misfold[r]' created\n",
      "unknown metabolite 'P15248-M5-misfold[r]' created\n",
      "unknown metabolite 'P15248-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P15248-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P15248-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P15248-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P15248-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P15248-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P15248-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P15248-G2M9[r]' created\n",
      "unknown metabolite 'P15248-preG1M9[r]' created\n",
      "unknown metabolite 'P15248-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P15248-G1M9[r]' created\n",
      "unknown metabolite 'P15248-M9[r]' created\n",
      "unknown metabolite 'P15248-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P15248-M8B[r]' created\n",
      "unknown metabolite 'P15248-M8B[g]' created\n",
      "unknown metabolite 'P15248-M7[g]' created\n",
      "unknown metabolite 'P15248-M6[g]' created\n",
      "unknown metabolite 'P15248-M5[g]' created\n",
      "unknown metabolite 'P15248-M5-GN[g]' created\n",
      "unknown metabolite 'P15248-M4-GN[g]' created\n",
      "unknown metabolite 'P15248-M3-GN[g]' created\n",
      "unknown metabolite 'P15248-M3-GN2[g]' created\n",
      "unknown metabolite 'P15248-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P15248-COPII-coated[c]' created\n",
      "unknown metabolite 'P15248-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P15248-preSV[g]' created\n",
      "unknown metabolite 'P15248-AP-coated[c]' created"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "unknown metabolite 'P15248_mature[sv]' created\n",
      "unknown metabolite 'P15248_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P22301[c]' created\n",
      "unknown metabolite 'P22301-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P22301-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P22301-BiP-adp[r]' created\n",
      "unknown metabolite 'P22301[r]' created\n",
      "unknown metabolite 'P22301-SRP[c]' created\n",
      "unknown metabolite 'P22301-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P22301-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P22301-SEC61C[c]' created\n",
      "unknown metabolite 'P22301-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P22301-sp[r]' created\n",
      "unknown metabolite 'P22301_preDSB[r]' created\n",
      "unknown metabolite 'P22301-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P22301_DSB[r]' created\n",
      "unknown metabolite 'P22301_preNG[r]' created\n",
      "unknown metabolite 'P22301-G3M9[r]' created\n",
      "unknown metabolite 'P22301-M9-misfold[r]' created\n",
      "unknown metabolite 'P22301-M8C-misfold[r]' created\n",
      "unknown metabolite 'P22301-M7A-misfold[r]' created\n",
      "unknown metabolite 'P22301-M6-misfold[r]' created\n",
      "unknown metabolite 'P22301-M5-misfold[r]' created\n",
      "unknown metabolite 'P22301-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P22301-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P22301-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P22301-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P22301-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P22301-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P22301-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P22301-G2M9[r]' created\n",
      "unknown metabolite 'P22301-preG1M9[r]' created\n",
      "unknown metabolite 'P22301-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P22301-G1M9[r]' created\n",
      "unknown metabolite 'P22301-M9[r]' created\n",
      "unknown metabolite 'P22301-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P22301-M8B[r]' created\n",
      "unknown metabolite 'P22301-M8B[g]' created\n",
      "unknown metabolite 'P22301-M7[g]' created\n",
      "unknown metabolite 'P22301-M6[g]' created\n",
      "unknown metabolite 'P22301-M5[g]' created\n",
      "unknown metabolite 'P22301-M5-GN[g]' created\n",
      "unknown metabolite 'P22301-M4-GN[g]' created\n",
      "unknown metabolite 'P22301-M3-GN[g]' created\n",
      "unknown metabolite 'P22301-M3-GN2[g]' created\n",
      "unknown metabolite 'P22301-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P22301-COPII-coated[c]' created\n",
      "unknown metabolite 'P22301-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P22301-preSV[g]' created\n",
      "unknown metabolite 'P22301-AP-coated[c]' created\n",
      "unknown metabolite 'P22301_mature[sv]' created\n",
      "unknown metabolite 'P22301_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P29459[c]' created\n",
      "unknown metabolite 'P29459-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P29459-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P29459-BiP-adp[r]' created\n",
      "unknown metabolite 'P29459[r]' created\n",
      "unknown metabolite 'P29459-SRP[c]' created\n",
      "unknown metabolite 'P29459-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P29459-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P29459-SEC61C[c]' created\n",
      "unknown metabolite 'P29459-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P29459-sp[r]' created\n",
      "unknown metabolite 'P29459_preDSB[r]' created\n",
      "unknown metabolite 'P29459-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P29459_DSB[r]' created\n",
      "unknown metabolite 'P29459_preNG[r]' created\n",
      "unknown metabolite 'P29459-G3M9[r]' created\n",
      "unknown metabolite 'P29459-M9-misfold[r]' created\n",
      "unknown metabolite 'P29459-M8C-misfold[r]' created\n",
      "unknown metabolite 'P29459-M7A-misfold[r]' created\n",
      "unknown metabolite 'P29459-M6-misfold[r]' created\n",
      "unknown metabolite 'P29459-M5-misfold[r]' created\n",
      "unknown metabolite 'P29459-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P29459-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P29459-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P29459-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P29459-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P29459-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P29459-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P29459-G2M9[r]' created\n",
      "unknown metabolite 'P29459-preG1M9[r]' created\n",
      "unknown metabolite 'P29459-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P29459-G1M9[r]' created\n",
      "unknown metabolite 'P29459-M9[r]' created\n",
      "unknown metabolite 'P29459-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P29459-M8B[r]' created\n",
      "unknown metabolite 'P29459-M8B[g]' created\n",
      "unknown metabolite 'P29459-M7[g]' created\n",
      "unknown metabolite 'P29459-M6[g]' created\n",
      "unknown metabolite 'P29459-M5[g]' created\n",
      "unknown metabolite 'P29459-M5-GN[g]' created\n",
      "unknown metabolite 'P29459-M4-GN[g]' created\n",
      "unknown metabolite 'P29459-M3-GN[g]' created\n",
      "unknown metabolite 'P29459-M3-GN2[g]' created\n",
      "unknown metabolite 'P29459-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P29459-COPII-coated[c]' created"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "unknown metabolite 'P29459-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P29459-preSV[g]' created\n",
      "unknown metabolite 'P29459-AP-coated[c]' created\n",
      "unknown metabolite 'P29459_mature[sv]' created\n",
      "unknown metabolite 'P29459_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P29460[c]' created\n",
      "unknown metabolite 'P29460-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P29460-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P29460-BiP-adp[r]' created\n",
      "unknown metabolite 'P29460[r]' created\n",
      "unknown metabolite 'P29460-SRP[c]' created\n",
      "unknown metabolite 'P29460-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P29460-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P29460-SEC61C[c]' created\n",
      "unknown metabolite 'P29460-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P29460-sp[r]' created\n",
      "unknown metabolite 'P29460_preDSB[r]' created\n",
      "unknown metabolite 'P29460-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P29460_DSB[r]' created\n",
      "unknown metabolite 'P29460_preNG[r]' created\n",
      "unknown metabolite 'P29460-G3M9[r]' created\n",
      "unknown metabolite 'P29460-M9-misfold[r]' created\n",
      "unknown metabolite 'P29460-M8C-misfold[r]' created\n",
      "unknown metabolite 'P29460-M7A-misfold[r]' created\n",
      "unknown metabolite 'P29460-M6-misfold[r]' created\n",
      "unknown metabolite 'P29460-M5-misfold[r]' created\n",
      "unknown metabolite 'P29460-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P29460-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P29460-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P29460-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P29460-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P29460-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P29460-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P29460-G2M9[r]' created\n",
      "unknown metabolite 'P29460-preG1M9[r]' created\n",
      "unknown metabolite 'P29460-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P29460-G1M9[r]' created\n",
      "unknown metabolite 'P29460-M9[r]' created\n",
      "unknown metabolite 'P29460-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P29460-M8B[r]' created\n",
      "unknown metabolite 'P29460-M8B[g]' created\n",
      "unknown metabolite 'P29460-M7[g]' created\n",
      "unknown metabolite 'P29460-M6[g]' created\n",
      "unknown metabolite 'P29460-M5[g]' created\n",
      "unknown metabolite 'P29460-M5-GN[g]' created\n",
      "unknown metabolite 'P29460-M4-GN[g]' created\n",
      "unknown metabolite 'P29460-M3-GN[g]' created\n",
      "unknown metabolite 'P29460-M3-GN2[g]' created\n",
      "unknown metabolite 'P29460-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P29460-COPII-coated[c]' created\n",
      "unknown metabolite 'P29460-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P29460-preSV[g]' created\n",
      "unknown metabolite 'P29460-AP-coated[c]' created\n",
      "unknown metabolite 'P29460_mature[sv]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P29460_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P35225[c]' created\n",
      "unknown metabolite 'P35225-BAT3-CALM[c]' created\n",
      "unknown metabolite 'P35225-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P35225-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P35225-BiP-adp[r]' created\n",
      "unknown metabolite 'P35225[r]' created\n",
      "unknown metabolite 'P35225-SRP[c]' created\n",
      "unknown metabolite 'P35225-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P35225-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P35225-SEC61C[c]' created\n",
      "unknown metabolite 'P35225-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P35225-sp[r]' created\n",
      "unknown metabolite 'P35225_preDSB[r]' created\n",
      "unknown metabolite 'P35225-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P35225_DSB[r]' created\n",
      "unknown metabolite 'P35225_preNG[r]' created\n",
      "unknown metabolite 'P35225-G3M9[r]' created\n",
      "unknown metabolite 'P35225-M9-misfold[r]' created\n",
      "unknown metabolite 'P35225-M8C-misfold[r]' created\n",
      "unknown metabolite 'P35225-M7A-misfold[r]' created\n",
      "unknown metabolite 'P35225-M6-misfold[r]' created\n",
      "unknown metabolite 'P35225-M5-misfold[r]' created\n",
      "unknown metabolite 'P35225-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P35225-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P35225-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P35225-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P35225-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P35225-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P35225-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P35225-G2M9[r]' created\n",
      "unknown metabolite 'P35225-preG1M9[r]' created\n",
      "unknown metabolite 'P35225-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P35225-G1M9[r]' created\n",
      "unknown metabolite 'P35225-M9[r]' created\n",
      "unknown metabolite 'P35225-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P35225-M8B[r]' created\n",
      "unknown metabolite 'P35225-M8B[g]' created\n",
      "unknown metabolite 'P35225-M7[g]' created\n",
      "unknown metabolite 'P35225-M6[g]' created\n",
      "unknown metabolite 'P35225-M5[g]' created\n",
      "unknown metabolite 'P35225-M5-GN[g]' created\n",
      "unknown metabolite 'P35225-M4-GN[g]' created\n",
      "unknown metabolite 'P35225-M3-GN[g]' created\n",
      "unknown metabolite 'P35225-M3-GN2[g]' created\n",
      "unknown metabolite 'P35225-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P35225-COPII-coated[c]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P35225-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P35225-preSV[g]' created\n",
      "unknown metabolite 'P35225-AP-coated[c]' created\n",
      "unknown metabolite 'P35225_mature[sv]' created\n",
      "unknown metabolite 'P35225_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P40933[c]' created\n",
      "unknown metabolite 'P40933-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P40933-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P40933-BiP-adp[r]' created\n",
      "unknown metabolite 'P40933[r]' created\n",
      "unknown metabolite 'P40933-SRP[c]' created\n",
      "unknown metabolite 'P40933-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P40933-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P40933-SEC61C[c]' created\n",
      "unknown metabolite 'P40933-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P40933-sp[r]' created\n",
      "unknown metabolite 'P40933_preDSB[r]' created\n",
      "unknown metabolite 'P40933-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P40933_DSB[r]' created\n",
      "unknown metabolite 'P40933_preNG[r]' created\n",
      "unknown metabolite 'P40933-G3M9[r]' created\n",
      "unknown metabolite 'P40933-M9-misfold[r]' created\n",
      "unknown metabolite 'P40933-M8C-misfold[r]' created\n",
      "unknown metabolite 'P40933-M7A-misfold[r]' created\n",
      "unknown metabolite 'P40933-M6-misfold[r]' created\n",
      "unknown metabolite 'P40933-M5-misfold[r]' created\n",
      "unknown metabolite 'P40933-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P40933-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P40933-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P40933-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P40933-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P40933-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P40933-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P40933-G2M9[r]' created\n",
      "unknown metabolite 'P40933-preG1M9[r]' created\n",
      "unknown metabolite 'P40933-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P40933-G1M9[r]' created\n",
      "unknown metabolite 'P40933-M9[r]' created\n",
      "unknown metabolite 'P40933-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P40933-M8B[r]' created\n",
      "unknown metabolite 'P40933-M8B[g]' created\n",
      "unknown metabolite 'P40933-M7[g]' created\n",
      "unknown metabolite 'P40933-M6[g]' created\n",
      "unknown metabolite 'P40933-M5[g]' created\n",
      "unknown metabolite 'P40933-M5-GN[g]' created\n",
      "unknown metabolite 'P40933-M4-GN[g]' created\n",
      "unknown metabolite 'P40933-M3-GN[g]' created\n",
      "unknown metabolite 'P40933-M3-GN2[g]' created\n",
      "unknown metabolite 'P40933-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P40933-COPII-coated[c]' created\n",
      "unknown metabolite 'P40933-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P40933-preSV[g]' created"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "unknown metabolite 'P40933-AP-coated[c]' created\n",
      "unknown metabolite 'P40933_mature[sv]' created\n",
      "unknown metabolite 'P40933_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'Q96F46[c]' created\n",
      "unknown metabolite 'Q96F46-ASNA1-atp[c]' created\n",
      "unknown metabolite 'Q96F46-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'Q96F46-BiP-adp[r]' created\n",
      "unknown metabolite 'Q96F46[r]' created\n",
      "unknown metabolite 'Q96F46-ASNA1-atp-WRB[r]' created\n",
      "unknown metabolite 'Q96F46_preDSB[r]' created\n",
      "unknown metabolite 'Q96F46-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'Q96F46_DSB[r]' created\n",
      "unknown metabolite 'Q96F46_preNG[r]' created\n",
      "unknown metabolite 'Q96F46-G3M9[r]' created\n",
      "unknown metabolite 'Q96F46-M9-misfold[r]' created\n",
      "unknown metabolite 'Q96F46-M8C-misfold[r]' created\n",
      "unknown metabolite 'Q96F46-M7A-misfold[r]' created\n",
      "unknown metabolite 'Q96F46-M6-misfold[r]' created\n",
      "unknown metabolite 'Q96F46-M5-misfold[r]' created\n",
      "unknown metabolite 'Q96F46-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'Q96F46-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'Q96F46-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'Q96F46-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'Q96F46-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'Q96F46-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'Q96F46-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'Q96F46-G2M9[r]' created\n",
      "unknown metabolite 'Q96F46-preG1M9[r]' created\n",
      "unknown metabolite 'Q96F46-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'Q96F46-G1M9[r]' created\n",
      "unknown metabolite 'Q96F46-M9[r]' created\n",
      "unknown metabolite 'Q96F46-M9-UGGT1[r]' created\n",
      "unknown metabolite 'Q96F46-M8B[r]' created\n",
      "unknown metabolite 'Q96F46-M8B[g]' created\n",
      "unknown metabolite 'Q96F46-M7[g]' created\n",
      "unknown metabolite 'Q96F46-M6[g]' created\n",
      "unknown metabolite 'Q96F46-M5[g]' created\n",
      "unknown metabolite 'Q96F46-M5-GN[g]' created\n",
      "unknown metabolite 'Q96F46-M4-GN[g]' created\n",
      "unknown metabolite 'Q96F46-M3-GN[g]' created\n",
      "unknown metabolite 'Q96F46-M3-GN2[g]' created\n",
      "unknown metabolite 'Q96F46-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'Q96F46-COPII-coated[c]' created\n",
      "unknown metabolite 'Q96F46-COPII-uncoated[c]' created\n",
      "unknown metabolite 'Q96F46-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'Q96F46-AP-coated[c]' created\n",
      "unknown metabolite 'Q96F46_mature[sv]' created\n",
      "unknown metabolite 'Q96F46_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'Q9Y2Y9[c]' created\n",
      "unknown metabolite 'P09919[c]' created\n",
      "unknown metabolite 'P09919-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P09919-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P09919-BiP-adp[r]' created\n",
      "unknown metabolite 'P09919[r]' created\n",
      "unknown metabolite 'P09919-SRP[c]' created\n",
      "unknown metabolite 'P09919-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P09919-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P09919-SEC61C[c]' created\n",
      "unknown metabolite 'P09919-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P09919-sp[r]' created\n",
      "unknown metabolite 'P09919_preDSB[r]' created\n",
      "unknown metabolite 'P09919-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P09919_DSB[r]' created\n",
      "unknown metabolite 'P09919-LMAN1L[r]' created\n",
      "unknown metabolite 'P09919-COPII-coated[c]' created\n",
      "unknown metabolite 'P09919-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P09919[g]' created\n",
      "unknown metabolite 'P09919_preOG[g]' created\n",
      "unknown metabolite 'P09919-acgal[g]' created\n",
      "unknown metabolite 'P09919-Core1[g]' created\n",
      "unknown metabolite 'P09919-Core2[g]' created\n",
      "unknown metabolite 'P09919-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P09919-AP-coated[c]' created\n",
      "unknown metabolite 'P09919_mature[sv]' created\n",
      "unknown metabolite 'P09919_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P02778[c]' created\n",
      "unknown metabolite 'P02778-BAT3-CALM[c]' created\n",
      "unknown metabolite 'P02778-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P02778-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P02778-BiP-adp[r]' created\n",
      "unknown metabolite 'P02778[r]' created\n",
      "unknown metabolite 'P02778-SRP[c]' created\n",
      "unknown metabolite 'P02778-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P02778-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P02778-SEC61C[c]' created\n",
      "unknown metabolite 'P02778-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P02778-sp[r]' created\n",
      "unknown metabolite 'P02778_preDSB[r]' created\n",
      "unknown metabolite 'P02778-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P02778_DSB[r]' created\n",
      "unknown metabolite 'P02778-LMAN1L[r]' created\n",
      "unknown metabolite 'P02778-COPII-coated[c]' created\n",
      "unknown metabolite 'P02778-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P02778[g]' created\n",
      "unknown metabolite 'P02778-preSV[g]' created\n",
      "unknown metabolite 'P02778-AP-coated[c]' created\n",
      "unknown metabolite 'P02778_mature[sv]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P02778_mature[e]' created\n",
      "unknown metabolite 'P13500[c]' created\n",
      "unknown metabolite 'P13500-BAT3-CALM[c]' created\n",
      "unknown metabolite 'P13500-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P13500-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P13500-BiP-adp[r]' created\n",
      "unknown metabolite 'P13500[r]' created\n",
      "unknown metabolite 'P13500-SRP[c]' created\n",
      "unknown metabolite 'P13500-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P13500-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P13500-SEC61C[c]' created\n",
      "unknown metabolite 'P13500-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P13500-sp[r]' created\n",
      "unknown metabolite 'P13500_preDSB[r]' created\n",
      "unknown metabolite 'P13500-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P13500_DSB[r]' created\n",
      "unknown metabolite 'P13500_preNG[r]' created\n",
      "unknown metabolite 'P13500-G3M9[r]' created\n",
      "unknown metabolite 'P13500-M9-misfold[r]' created\n",
      "unknown metabolite 'P13500-M8C-misfold[r]' created\n",
      "unknown metabolite 'P13500-M7A-misfold[r]' created\n",
      "unknown metabolite 'P13500-M6-misfold[r]' created\n",
      "unknown metabolite 'P13500-M5-misfold[r]' created\n",
      "unknown metabolite 'P13500-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P13500-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P13500-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P13500-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P13500-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P13500-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P13500-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P13500-G2M9[r]' created\n",
      "unknown metabolite 'P13500-preG1M9[r]' created\n",
      "unknown metabolite 'P13500-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P13500-G1M9[r]' created\n",
      "unknown metabolite 'P13500-M9[r]' created\n",
      "unknown metabolite 'P13500-M9-UGGT1[r]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P13500-M8B[r]' created\n",
      "unknown metabolite 'P13500-M8B[g]' created\n",
      "unknown metabolite 'P13500-M7[g]' created\n",
      "unknown metabolite 'P13500-M6[g]' created\n",
      "unknown metabolite 'P13500-M5[g]' created\n",
      "unknown metabolite 'P13500-M5-GN[g]' created\n",
      "unknown metabolite 'P13500-M4-GN[g]' created\n",
      "unknown metabolite 'P13500-M3-GN[g]' created\n",
      "unknown metabolite 'P13500-M3-GN2[g]' created\n",
      "unknown metabolite 'P13500-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P13500-COPII-coated[c]' created\n",
      "unknown metabolite 'P13500-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P13500-preSV[g]' created\n",
      "unknown metabolite 'P13500-AP-coated[c]' created\n",
      "unknown metabolite 'P13500_mature[sv]' created\n",
      "unknown metabolite 'P13500_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P10147[c]' created\n",
      "unknown metabolite 'P10147-BAT3-CALM[c]' created\n",
      "unknown metabolite 'P10147-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P10147-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P10147-BiP-adp[r]' created\n",
      "unknown metabolite 'P10147[r]' created\n",
      "unknown metabolite 'P10147-SRP[c]' created\n",
      "unknown metabolite 'P10147-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P10147-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P10147-SEC61C[c]' created\n",
      "unknown metabolite 'P10147-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P10147-sp[r]' created\n",
      "unknown metabolite 'P10147_preDSB[r]' created\n",
      "unknown metabolite 'P10147-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P10147_DSB[r]' created\n",
      "unknown metabolite 'P10147-LMAN1L[r]' created\n",
      "unknown metabolite 'P10147-COPII-coated[c]' created\n",
      "unknown metabolite 'P10147-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P10147[g]' created\n",
      "unknown metabolite 'P10147-preSV[g]' created\n",
      "unknown metabolite 'P10147-AP-coated[c]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P10147_mature[sv]' created\n",
      "unknown metabolite 'P10147_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P01375[c]' created\n",
      "unknown metabolite 'P15692[c]' created\n",
      "unknown metabolite 'P15692-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P15692-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P15692-BiP-adp[r]' created\n",
      "unknown metabolite 'P15692[r]' created\n",
      "unknown metabolite 'P15692-SRP[c]' created\n",
      "unknown metabolite 'P15692-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P15692-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P15692-SEC61C[c]' created\n",
      "unknown metabolite 'P15692-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P15692-sp[r]' created\n",
      "unknown metabolite 'P15692_preDSB[r]' created\n",
      "unknown metabolite 'P15692-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P15692_DSB[r]' created\n",
      "unknown metabolite 'P15692_preNG[r]' created\n",
      "unknown metabolite 'P15692-G3M9[r]' created\n",
      "unknown metabolite 'P15692-M9-misfold[r]' created\n",
      "unknown metabolite 'P15692-M8C-misfold[r]' created\n",
      "unknown metabolite 'P15692-M7A-misfold[r]' created\n",
      "unknown metabolite 'P15692-M6-misfold[r]' created\n",
      "unknown metabolite 'P15692-M5-misfold[r]' created\n",
      "unknown metabolite 'P15692-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P15692-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P15692-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P15692-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P15692-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P15692-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P15692-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P15692-G2M9[r]' created\n",
      "unknown metabolite 'P15692-preG1M9[r]' created\n",
      "unknown metabolite 'P15692-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P15692-G1M9[r]' created\n",
      "unknown metabolite 'P15692-M9[r]' created\n",
      "unknown metabolite 'P15692-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P15692-M8B[r]' created\n",
      "unknown metabolite 'P15692-M8B[g]' created\n",
      "unknown metabolite 'P15692-M7[g]' created\n",
      "unknown metabolite 'P15692-M6[g]' created\n",
      "unknown metabolite 'P15692-M5[g]' created\n",
      "unknown metabolite 'P15692-M5-GN[g]' created\n",
      "unknown metabolite 'P15692-M4-GN[g]' created\n",
      "unknown metabolite 'P15692-M3-GN[g]' created\n",
      "unknown metabolite 'P15692-M3-GN2[g]' created\n",
      "unknown metabolite 'P15692-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P15692-COPII-coated[c]' created\n",
      "unknown metabolite 'P15692-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P15692-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P15692-AP-coated[c]' created\n",
      "unknown metabolite 'P15692_mature[sv]' created\n",
      "unknown metabolite 'P15692_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P01127[c]' created\n",
      "unknown metabolite 'P01127-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P01127-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P01127-BiP-adp[r]' created\n",
      "unknown metabolite 'P01127[r]' created\n",
      "unknown metabolite 'P01127-SRP[c]' created\n",
      "unknown metabolite 'P01127-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P01127-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P01127-SEC61C[c]' created\n",
      "unknown metabolite 'P01127-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P01127-sp[r]' created\n",
      "unknown metabolite 'P01127_preDSB[r]' created\n",
      "unknown metabolite 'P01127-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P01127_DSB[r]' created\n",
      "unknown metabolite 'P01127_preNG[r]' created\n",
      "unknown metabolite 'P01127-G3M9[r]' created\n",
      "unknown metabolite 'P01127-M9-misfold[r]' created\n",
      "unknown metabolite 'P01127-M8C-misfold[r]' created\n",
      "unknown metabolite 'P01127-M7A-misfold[r]' created\n",
      "unknown metabolite 'P01127-M6-misfold[r]' created\n",
      "unknown metabolite 'P01127-M5-misfold[r]' created\n",
      "unknown metabolite 'P01127-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P01127-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P01127-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P01127-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P01127-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P01127-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P01127-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P01127-G2M9[r]' created\n",
      "unknown metabolite 'P01127-preG1M9[r]' created\n",
      "unknown metabolite 'P01127-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P01127-G1M9[r]' created\n",
      "unknown metabolite 'P01127-M9[r]' created\n",
      "unknown metabolite 'P01127-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P01127-M8B[r]' created\n",
      "unknown metabolite 'P01127-M8B[g]' created\n",
      "unknown metabolite 'P01127-M7[g]' created\n",
      "unknown metabolite 'P01127-M6[g]' created\n",
      "unknown metabolite 'P01127-M5[g]' created\n",
      "unknown metabolite 'P01127-M5-GN[g]' created\n",
      "unknown metabolite 'P01127-M4-GN[g]' created\n",
      "unknown metabolite 'P01127-M3-GN[g]' created\n",
      "unknown metabolite 'P01127-M3-GN2[g]' created\n",
      "unknown metabolite 'P01127-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P01127-COPII-coated[c]' created\n",
      "unknown metabolite 'P01127-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P01127-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P01127-AP-coated[c]' created\n",
      "unknown metabolite 'P01127_mature[sv]' created\n",
      "unknown metabolite 'P01127_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P13236[c]' created\n",
      "unknown metabolite 'P13236-BAT3-CALM[c]' created\n",
      "unknown metabolite 'P13236-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P13236-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P13236-BiP-adp[r]' created\n",
      "unknown metabolite 'P13236[r]' created\n",
      "unknown metabolite 'P13236-SRP[c]' created\n",
      "unknown metabolite 'P13236-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P13236-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P13236-SEC61C[c]' created\n",
      "unknown metabolite 'P13236-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P13236-sp[r]' created\n",
      "unknown metabolite 'P13236_preDSB[r]' created\n",
      "unknown metabolite 'P13236-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P13236_DSB[r]' created\n",
      "unknown metabolite 'P13236-LMAN1L[r]' created\n",
      "unknown metabolite 'P13236-COPII-coated[c]' created\n",
      "unknown metabolite 'P13236-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P13236[g]' created\n",
      "unknown metabolite 'P13236-preSV[g]' created\n",
      "unknown metabolite 'P13236-AP-coated[c]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P13236_mature[sv]' created\n",
      "unknown metabolite 'P13236_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P04141[c]' created\n",
      "unknown metabolite 'P04141-BAT3-CALM[c]' created\n",
      "unknown metabolite 'P04141-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P04141-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P04141-BiP-adp[r]' created\n",
      "unknown metabolite 'P04141[r]' created\n",
      "unknown metabolite 'P04141-SRP[c]' created\n",
      "unknown metabolite 'P04141-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P04141-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P04141-SEC61C[c]' created\n",
      "unknown metabolite 'P04141-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P04141-sp[r]' created\n",
      "unknown metabolite 'P04141_preDSB[r]' created\n",
      "unknown metabolite 'P04141-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P04141_DSB[r]' created\n",
      "unknown metabolite 'P04141_preNG[r]' created\n",
      "unknown metabolite 'P04141-G3M9[r]' created\n",
      "unknown metabolite 'P04141-M9-misfold[r]' created\n",
      "unknown metabolite 'P04141-M8C-misfold[r]' created\n",
      "unknown metabolite 'P04141-M7A-misfold[r]' created\n",
      "unknown metabolite 'P04141-M6-misfold[r]' created\n",
      "unknown metabolite 'P04141-M5-misfold[r]' created\n",
      "unknown metabolite 'P04141-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P04141-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P04141-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P04141-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P04141-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P04141-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P04141-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P04141-G2M9[r]' created\n",
      "unknown metabolite 'P04141-preG1M9[r]' created\n",
      "unknown metabolite 'P04141-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P04141-G1M9[r]' created\n",
      "unknown metabolite 'P04141-M9[r]' created\n",
      "unknown metabolite 'P04141-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P04141-M8B[r]' created\n",
      "unknown metabolite 'P04141-M8B[g]' created\n",
      "unknown metabolite 'P04141-M7[g]' created\n",
      "unknown metabolite 'P04141-M6[g]' created\n",
      "unknown metabolite 'P04141-M5[g]' created\n",
      "unknown metabolite 'P04141-M5-GN[g]' created\n",
      "unknown metabolite 'P04141-M4-GN[g]' created\n",
      "unknown metabolite 'P04141-M3-GN[g]' created\n",
      "unknown metabolite 'P04141-M3-GN2[g]' created\n",
      "unknown metabolite 'P04141-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P04141-COPII-coated[c]' created\n",
      "unknown metabolite 'P04141-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P04141_preOG[g]' created\n",
      "unknown metabolite 'P04141-acgal[g]' created\n",
      "unknown metabolite 'P04141-Core1[g]' created\n",
      "unknown metabolite 'P04141-Core2[g]' created\n",
      "unknown metabolite 'P04141-preSV[g]' created\n",
      "unknown metabolite 'P04141-AP-coated[c]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P04141_mature[sv]' created\n",
      "unknown metabolite 'P04141_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P09038[c]' created\n",
      "unknown metabolite 'P51671[c]' created\n",
      "unknown metabolite 'P51671-BAT3-CALM[c]' created\n",
      "unknown metabolite 'P51671-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P51671-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P51671-BiP-adp[r]' created\n",
      "unknown metabolite 'P51671[r]' created\n",
      "unknown metabolite 'P51671-SRP[c]' created\n",
      "unknown metabolite 'P51671-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P51671-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P51671-SEC61C[c]' created\n",
      "unknown metabolite 'P51671-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P51671-sp[r]' created\n",
      "unknown metabolite 'P51671_preDSB[r]' created\n",
      "unknown metabolite 'P51671-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P51671_DSB[r]' created\n",
      "unknown metabolite 'P51671-LMAN1L[r]' created\n",
      "unknown metabolite 'P51671-COPII-coated[c]' created\n",
      "unknown metabolite 'P51671-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P51671[g]' created\n",
      "unknown metabolite 'P51671_preOG[g]' created\n",
      "unknown metabolite 'P51671-acgal[g]' created\n",
      "unknown metabolite 'P51671-Core1[g]' created\n",
      "unknown metabolite 'P51671-Core2[g]' created\n",
      "unknown metabolite 'P51671-preSV[g]' created\n",
      "unknown metabolite 'P51671-AP-coated[c]' created\n",
      "unknown metabolite 'P51671_mature[sv]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P51671_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P78556[c]' created\n",
      "unknown metabolite 'P78556-BAT3-CALM[c]' created\n",
      "unknown metabolite 'P78556-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P78556-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P78556-BiP-adp[r]' created\n",
      "unknown metabolite 'P78556[r]' created\n",
      "unknown metabolite 'P78556-SRP[c]' created\n",
      "unknown metabolite 'P78556-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P78556-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P78556-SEC61C[c]' created\n",
      "unknown metabolite 'P78556-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P78556-sp[r]' created\n",
      "unknown metabolite 'P78556_preDSB[r]' created\n",
      "unknown metabolite 'P78556-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P78556_DSB[r]' created\n",
      "unknown metabolite 'P78556-LMAN1L[r]' created\n",
      "unknown metabolite 'P78556-COPII-coated[c]' created\n",
      "unknown metabolite 'P78556-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P78556[g]' created\n",
      "unknown metabolite 'P78556-preSV[g]' created\n",
      "unknown metabolite 'P78556-AP-coated[c]' created\n",
      "unknown metabolite 'P78556_mature[sv]' created\n",
      "unknown metabolite 'P78556_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'Q969D9[c]' created\n",
      "unknown metabolite 'Q969D9-BAT3-CALM[c]' created\n",
      "unknown metabolite 'Q969D9-ASNA1-atp[c]' created\n",
      "unknown metabolite 'Q969D9-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'Q969D9-BiP-adp[r]' created\n",
      "unknown metabolite 'Q969D9[r]' created\n",
      "unknown metabolite 'Q969D9-SRP[c]' created\n",
      "unknown metabolite 'Q969D9-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'Q969D9-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'Q969D9-SEC61C[c]' created\n",
      "unknown metabolite 'Q969D9-SEC61-SPC[r]' created\n",
      "unknown metabolite 'Q969D9-sp[r]' created\n",
      "unknown metabolite 'Q969D9_preDSB[r]' created\n",
      "unknown metabolite 'Q969D9-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'Q969D9_DSB[r]' created\n",
      "unknown metabolite 'Q969D9_preNG[r]' created\n",
      "unknown metabolite 'Q969D9-G3M9[r]' created\n",
      "unknown metabolite 'Q969D9-M9-misfold[r]' created\n",
      "unknown metabolite 'Q969D9-M8C-misfold[r]' created\n",
      "unknown metabolite 'Q969D9-M7A-misfold[r]' created\n",
      "unknown metabolite 'Q969D9-M6-misfold[r]' created\n",
      "unknown metabolite 'Q969D9-M5-misfold[r]' created\n",
      "unknown metabolite 'Q969D9-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'Q969D9-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'Q969D9-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'Q969D9-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'Q969D9-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'Q969D9-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'Q969D9-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'Q969D9-G2M9[r]' created\n",
      "unknown metabolite 'Q969D9-preG1M9[r]' created\n",
      "unknown metabolite 'Q969D9-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'Q969D9-G1M9[r]' created\n",
      "unknown metabolite 'Q969D9-M9[r]' created\n",
      "unknown metabolite 'Q969D9-M9-UGGT1[r]' created\n",
      "unknown metabolite 'Q969D9-M8B[r]' created\n",
      "unknown metabolite 'Q969D9-M8B[g]' created\n",
      "unknown metabolite 'Q969D9-M7[g]' created\n",
      "unknown metabolite 'Q969D9-M6[g]' created\n",
      "unknown metabolite 'Q969D9-M5[g]' created\n",
      "unknown metabolite 'Q969D9-M5-GN[g]' created\n",
      "unknown metabolite 'Q969D9-M4-GN[g]' created\n",
      "unknown metabolite 'Q969D9-M3-GN[g]' created\n",
      "unknown metabolite 'Q969D9-M3-GN2[g]' created\n",
      "unknown metabolite 'Q969D9-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'Q969D9-COPII-coated[c]' created\n",
      "unknown metabolite 'Q969D9-COPII-uncoated[c]' created\n",
      "unknown metabolite 'Q969D9-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'Q969D9-AP-coated[c]' created\n",
      "unknown metabolite 'Q969D9_mature[sv]' created\n",
      "unknown metabolite 'Q969D9_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'Q06830[c]' created\n",
      "unknown metabolite 'P32119[c]' created\n",
      "unknown metabolite 'Q16552[c]' created\n",
      "unknown metabolite 'Q16552-BAT3-CALM[c]' created\n",
      "unknown metabolite 'Q16552-ASNA1-atp[c]' created\n",
      "unknown metabolite 'Q16552-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'Q16552-BiP-adp[r]' created\n",
      "unknown metabolite 'Q16552[r]' created\n",
      "unknown metabolite 'Q16552-SRP[c]' created\n",
      "unknown metabolite 'Q16552-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'Q16552-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'Q16552-SEC61C[c]' created\n",
      "unknown metabolite 'Q16552-SEC61-SPC[r]' created\n",
      "unknown metabolite 'Q16552-sp[r]' created\n",
      "unknown metabolite 'Q16552_preDSB[r]' created\n",
      "unknown metabolite 'Q16552-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'Q16552_DSB[r]' created\n",
      "unknown metabolite 'Q16552_preNG[r]' created\n",
      "unknown metabolite 'Q16552-G3M9[r]' created\n",
      "unknown metabolite 'Q16552-M9-misfold[r]' created\n",
      "unknown metabolite 'Q16552-M8C-misfold[r]' created\n",
      "unknown metabolite 'Q16552-M7A-misfold[r]' created\n",
      "unknown metabolite 'Q16552-M6-misfold[r]' created\n",
      "unknown metabolite 'Q16552-M5-misfold[r]' created\n",
      "unknown metabolite 'Q16552-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'Q16552-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'Q16552-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'Q16552-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'Q16552-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'Q16552-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'Q16552-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'Q16552-G2M9[r]' created\n",
      "unknown metabolite 'Q16552-preG1M9[r]' created\n",
      "unknown metabolite 'Q16552-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'Q16552-G1M9[r]' created\n",
      "unknown metabolite 'Q16552-M9[r]' created\n",
      "unknown metabolite 'Q16552-M9-UGGT1[r]' created\n",
      "unknown metabolite 'Q16552-M8B[r]' created\n",
      "unknown metabolite 'Q16552-M8B[g]' created\n",
      "unknown metabolite 'Q16552-M7[g]' created\n",
      "unknown metabolite 'Q16552-M6[g]' created\n",
      "unknown metabolite 'Q16552-M5[g]' created\n",
      "unknown metabolite 'Q16552-M5-GN[g]' created\n",
      "unknown metabolite 'Q16552-M4-GN[g]' created\n",
      "unknown metabolite 'Q16552-M3-GN[g]' created\n",
      "unknown metabolite 'Q16552-M3-GN2[g]' created\n",
      "unknown metabolite 'Q16552-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'Q16552-COPII-coated[c]' created\n",
      "unknown metabolite 'Q16552-COPII-uncoated[c]' created\n",
      "unknown metabolite 'Q16552-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'Q16552-AP-coated[c]' created\n",
      "unknown metabolite 'Q16552_mature[sv]' created\n",
      "unknown metabolite 'Q16552_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'Q96PD4[c]' created\n",
      "unknown metabolite 'Q96PD4-ASNA1-atp[c]' created\n",
      "unknown metabolite 'Q96PD4-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'Q96PD4-BiP-adp[r]' created\n",
      "unknown metabolite 'Q96PD4[r]' created\n",
      "unknown metabolite 'Q96PD4-SRP[c]' created\n",
      "unknown metabolite 'Q96PD4-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'Q96PD4-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'Q96PD4-SEC61C[c]' created\n",
      "unknown metabolite 'Q96PD4-SEC61-SPC[r]' created\n",
      "unknown metabolite 'Q96PD4-sp[r]' created\n",
      "unknown metabolite 'Q96PD4_preDSB[r]' created\n",
      "unknown metabolite 'Q96PD4-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'Q96PD4_DSB[r]' created\n",
      "unknown metabolite 'Q96PD4_preNG[r]' created\n",
      "unknown metabolite 'Q96PD4-G3M9[r]' created\n",
      "unknown metabolite 'Q96PD4-M9-misfold[r]' created\n",
      "unknown metabolite 'Q96PD4-M8C-misfold[r]' created\n",
      "unknown metabolite 'Q96PD4-M7A-misfold[r]' created\n",
      "unknown metabolite 'Q96PD4-M6-misfold[r]' created\n",
      "unknown metabolite 'Q96PD4-M5-misfold[r]' created\n",
      "unknown metabolite 'Q96PD4-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'Q96PD4-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'Q96PD4-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'Q96PD4-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'Q96PD4-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'Q96PD4-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'Q96PD4-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'Q96PD4-G2M9[r]' created\n",
      "unknown metabolite 'Q96PD4-preG1M9[r]' created\n",
      "unknown metabolite 'Q96PD4-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'Q96PD4-G1M9[r]' created\n",
      "unknown metabolite 'Q96PD4-M9[r]' created\n",
      "unknown metabolite 'Q96PD4-M9-UGGT1[r]' created\n",
      "unknown metabolite 'Q96PD4-M8B[r]' created\n",
      "unknown metabolite 'Q96PD4-M8B[g]' created\n",
      "unknown metabolite 'Q96PD4-M7[g]' created\n",
      "unknown metabolite 'Q96PD4-M6[g]' created\n",
      "unknown metabolite 'Q96PD4-M5[g]' created\n",
      "unknown metabolite 'Q96PD4-M5-GN[g]' created\n",
      "unknown metabolite 'Q96PD4-M4-GN[g]' created\n",
      "unknown metabolite 'Q96PD4-M3-GN[g]' created\n",
      "unknown metabolite 'Q96PD4-M3-GN2[g]' created\n",
      "unknown metabolite 'Q96PD4-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'Q96PD4-COPII-coated[c]' created\n",
      "unknown metabolite 'Q96PD4-COPII-uncoated[c]' created\n",
      "unknown metabolite 'Q96PD4-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'Q96PD4-AP-coated[c]' created\n",
      "unknown metabolite 'Q96PD4_mature[sv]' created\n",
      "unknown metabolite 'Q96PD4_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'Q8NAC3[c]' created\n",
      "unknown metabolite 'Q8NAC3-ASNA1-atp[c]' created\n",
      "unknown metabolite 'Q8NAC3-ASNA1-atp-WRB[r]' created\n",
      "unknown metabolite 'Q8NAC3[r]' created\n",
      "unknown metabolite 'Q8NAC3_preNG[r]' created\n",
      "unknown metabolite 'Q8NAC3-G3M9[r]' created\n",
      "unknown metabolite 'Q8NAC3-M9-misfold[r]' created\n",
      "unknown metabolite 'Q8NAC3-M8C-misfold[r]' created\n",
      "unknown metabolite 'Q8NAC3-M7A-misfold[r]' created\n",
      "unknown metabolite 'Q8NAC3-M6-misfold[r]' created\n",
      "unknown metabolite 'Q8NAC3-M5-misfold[r]' created\n",
      "unknown metabolite 'Q8NAC3-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'Q8NAC3-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'Q8NAC3-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'Q8NAC3-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'Q8NAC3-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'Q8NAC3-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'Q8NAC3-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'Q8NAC3-G2M9[r]' created\n",
      "unknown metabolite 'Q8NAC3-preG1M9[r]' created\n",
      "unknown metabolite 'Q8NAC3-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'Q8NAC3-G1M9[r]' created\n",
      "unknown metabolite 'Q8NAC3-M9[r]' created\n",
      "unknown metabolite 'Q8NAC3-M9-UGGT1[r]' created\n",
      "unknown metabolite 'Q8NAC3-M8B[r]' created\n",
      "unknown metabolite 'Q8NAC3-M8B[g]' created\n",
      "unknown metabolite 'Q8NAC3-M7[g]' created\n",
      "unknown metabolite 'Q8NAC3-M6[g]' created\n",
      "unknown metabolite 'Q8NAC3-M5[g]' created\n",
      "unknown metabolite 'Q8NAC3-M5-GN[g]' created\n",
      "unknown metabolite 'Q8NAC3-M4-GN[g]' created\n",
      "unknown metabolite 'Q8NAC3-M3-GN[g]' created\n",
      "unknown metabolite 'Q8NAC3-M3-GN2[g]' created\n",
      "unknown metabolite 'Q8NAC3-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'Q8NAC3-COPII-coated[c]' created\n",
      "unknown metabolite 'Q8NAC3-COPII-uncoated[c]' created\n",
      "unknown metabolite 'Q8NAC3-preSV[g]' created\n",
      "unknown metabolite 'Q8NAC3-AP-coated[c]' created\n",
      "unknown metabolite 'Q8NAC3_mature[sv]' created\n",
      "unknown metabolite 'Q8NAC3_mature[pm]' created"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "unknown metabolite 'P01009[c]' created\n",
      "unknown metabolite 'P01009-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P01009-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P01009-BiP-adp[r]' created\n",
      "unknown metabolite 'P01009[r]' created\n",
      "unknown metabolite 'P01009-SRP[c]' created\n",
      "unknown metabolite 'P01009-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P01009-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P01009-SEC61C[c]' created\n",
      "unknown metabolite 'P01009-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P01009-sp[r]' created\n",
      "unknown metabolite 'P01009_preNG[r]' created\n",
      "unknown metabolite 'P01009-G3M9[r]' created\n",
      "unknown metabolite 'P01009-M9-misfold[r]' created\n",
      "unknown metabolite 'P01009-M8C-misfold[r]' created\n",
      "unknown metabolite 'P01009-M7A-misfold[r]' created\n",
      "unknown metabolite 'P01009-M6-misfold[r]' created\n",
      "unknown metabolite 'P01009-M5-misfold[r]' created\n",
      "unknown metabolite 'P01009-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P01009-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P01009-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P01009-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P01009-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P01009-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P01009-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P01009-G2M9[r]' created\n",
      "unknown metabolite 'P01009-preG1M9[r]' created\n",
      "unknown metabolite 'P01009-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P01009-G1M9[r]' created\n",
      "unknown metabolite 'P01009-M9[r]' created\n",
      "unknown metabolite 'P01009-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P01009-M8B[r]' created\n",
      "unknown metabolite 'P01009-M8B[g]' created\n",
      "unknown metabolite 'P01009-M7[g]' created\n",
      "unknown metabolite 'P01009-M6[g]' created\n",
      "unknown metabolite 'P01009-M5[g]' created\n",
      "unknown metabolite 'P01009-M5-GN[g]' created\n",
      "unknown metabolite 'P01009-M4-GN[g]' created\n",
      "unknown metabolite 'P01009-M3-GN[g]' created\n",
      "unknown metabolite 'P01009-M3-GN2[g]' created\n",
      "unknown metabolite 'P01009-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P01009-COPII-coated[c]' created\n",
      "unknown metabolite 'P01009-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P01009-preSV[g]' created\n",
      "unknown metabolite 'P01009-AP-coated[c]' created\n",
      "unknown metabolite 'P01009_mature[sv]' created\n",
      "unknown metabolite 'P01009_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P02763[c]' created\n",
      "unknown metabolite 'P02763-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P02763-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P02763-BiP-adp[r]' created\n",
      "unknown metabolite 'P02763[r]' created\n",
      "unknown metabolite 'P02763-SRP[c]' created\n",
      "unknown metabolite 'P02763-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P02763-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P02763-SEC61C[c]' created\n",
      "unknown metabolite 'P02763-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P02763-sp[r]' created\n",
      "unknown metabolite 'P02763_preDSB[r]' created\n",
      "unknown metabolite 'P02763-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P02763_DSB[r]' created\n",
      "unknown metabolite 'P02763_preNG[r]' created\n",
      "unknown metabolite 'P02763-G3M9[r]' created\n",
      "unknown metabolite 'P02763-M9-misfold[r]' created\n",
      "unknown metabolite 'P02763-M8C-misfold[r]' created\n",
      "unknown metabolite 'P02763-M7A-misfold[r]' created\n",
      "unknown metabolite 'P02763-M6-misfold[r]' created\n",
      "unknown metabolite 'P02763-M5-misfold[r]' created\n",
      "unknown metabolite 'P02763-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P02763-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P02763-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P02763-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P02763-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P02763-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P02763-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P02763-G2M9[r]' created\n",
      "unknown metabolite 'P02763-preG1M9[r]' created\n",
      "unknown metabolite 'P02763-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P02763-G1M9[r]' created\n",
      "unknown metabolite 'P02763-M9[r]' created\n",
      "unknown metabolite 'P02763-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P02763-M8B[r]' created\n",
      "unknown metabolite 'P02763-M8B[g]' created\n",
      "unknown metabolite 'P02763-M7[g]' created\n",
      "unknown metabolite 'P02763-M6[g]' created\n",
      "unknown metabolite 'P02763-M5[g]' created\n",
      "unknown metabolite 'P02763-M5-GN[g]' created\n",
      "unknown metabolite 'P02763-M4-GN[g]' created\n",
      "unknown metabolite 'P02763-M3-GN[g]' created\n",
      "unknown metabolite 'P02763-M3-GN2[g]' created\n",
      "unknown metabolite 'P02763-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P02763-COPII-coated[c]' created\n",
      "unknown metabolite 'P02763-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P02763-preSV[g]' created\n",
      "unknown metabolite 'P02763-AP-coated[c]' created\n",
      "unknown metabolite 'P02763_mature[sv]' created"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "unknown metabolite 'P02763_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P00738[c]' created\n",
      "unknown metabolite 'P00738-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P00738-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P00738-BiP-adp[r]' created\n",
      "unknown metabolite 'P00738[r]' created\n",
      "unknown metabolite 'P00738-SRP[c]' created\n",
      "unknown metabolite 'P00738-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P00738-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P00738-SEC61C[c]' created\n",
      "unknown metabolite 'P00738-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P00738-sp[r]' created\n",
      "unknown metabolite 'P00738_preDSB[r]' created\n",
      "unknown metabolite 'P00738-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P00738_DSB[r]' created\n",
      "unknown metabolite 'P00738_preNG[r]' created\n",
      "unknown metabolite 'P00738-G3M9[r]' created\n",
      "unknown metabolite 'P00738-M9-misfold[r]' created\n",
      "unknown metabolite 'P00738-M8C-misfold[r]' created\n",
      "unknown metabolite 'P00738-M7A-misfold[r]' created\n",
      "unknown metabolite 'P00738-M6-misfold[r]' created\n",
      "unknown metabolite 'P00738-M5-misfold[r]' created\n",
      "unknown metabolite 'P00738-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P00738-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P00738-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P00738-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P00738-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P00738-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P00738-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P00738-G2M9[r]' created\n",
      "unknown metabolite 'P00738-preG1M9[r]' created\n",
      "unknown metabolite 'P00738-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P00738-G1M9[r]' created\n",
      "unknown metabolite 'P00738-M9[r]' created\n",
      "unknown metabolite 'P00738-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P00738-M8B[r]' created\n",
      "unknown metabolite 'P00738-M8B[g]' created\n",
      "unknown metabolite 'P00738-M7[g]' created\n",
      "unknown metabolite 'P00738-M6[g]' created\n",
      "unknown metabolite 'P00738-M5[g]' created\n",
      "unknown metabolite 'P00738-M5-GN[g]' created\n",
      "unknown metabolite 'P00738-M4-GN[g]' created\n",
      "unknown metabolite 'P00738-M3-GN[g]' created\n",
      "unknown metabolite 'P00738-M3-GN2[g]' created\n",
      "unknown metabolite 'P00738-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P00738-COPII-coated[c]' created\n",
      "unknown metabolite 'P00738-COPII-uncoated[c]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P00738-preSV[g]' created\n",
      "unknown metabolite 'P00738-AP-coated[c]' created\n",
      "unknown metabolite 'P00738_mature[sv]' created\n",
      "unknown metabolite 'P00738_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P01011[c]' created\n",
      "unknown metabolite 'P01011-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P01011-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P01011-BiP-adp[r]' created\n",
      "unknown metabolite 'P01011[r]' created\n",
      "unknown metabolite 'P01011-SRP[c]' created\n",
      "unknown metabolite 'P01011-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P01011-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P01011-SEC61C[c]' created\n",
      "unknown metabolite 'P01011-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P01011-sp[r]' created\n",
      "unknown metabolite 'P01011_preNG[r]' created\n",
      "unknown metabolite 'P01011-G3M9[r]' created\n",
      "unknown metabolite 'P01011-M9-misfold[r]' created\n",
      "unknown metabolite 'P01011-M8C-misfold[r]' created\n",
      "unknown metabolite 'P01011-M7A-misfold[r]' created\n",
      "unknown metabolite 'P01011-M6-misfold[r]' created\n",
      "unknown metabolite 'P01011-M5-misfold[r]' created\n",
      "unknown metabolite 'P01011-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P01011-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P01011-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P01011-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P01011-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P01011-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P01011-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P01011-G2M9[r]' created\n",
      "unknown metabolite 'P01011-preG1M9[r]' created\n",
      "unknown metabolite 'P01011-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P01011-G1M9[r]' created\n",
      "unknown metabolite 'P01011-M9[r]' created\n",
      "unknown metabolite 'P01011-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P01011-M8B[r]' created\n",
      "unknown metabolite 'P01011-M8B[g]' created\n",
      "unknown metabolite 'P01011-M7[g]' created\n",
      "unknown metabolite 'P01011-M6[g]' created\n",
      "unknown metabolite 'P01011-M5[g]' created\n",
      "unknown metabolite 'P01011-M5-GN[g]' created\n",
      "unknown metabolite 'P01011-M4-GN[g]' created\n",
      "unknown metabolite 'P01011-M3-GN[g]' created\n",
      "unknown metabolite 'P01011-M3-GN2[g]' created\n",
      "unknown metabolite 'P01011-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P01011-COPII-coated[c]' created\n",
      "unknown metabolite 'P01011-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P01011-preSV[g]' created\n",
      "unknown metabolite 'P01011-AP-coated[c]' created\n",
      "unknown metabolite 'P01011_mature[sv]' created"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "unknown metabolite 'P01011_mature[e]' created\n",
      "unknown metabolite 'P02765[c]' created\n",
      "unknown metabolite 'P02765-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P02765-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P02765-BiP-adp[r]' created\n",
      "unknown metabolite 'P02765[r]' created\n",
      "unknown metabolite 'P02765-SRP[c]' created\n",
      "unknown metabolite 'P02765-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P02765-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P02765-SEC61C[c]' created\n",
      "unknown metabolite 'P02765-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P02765-sp[r]' created\n",
      "unknown metabolite 'P02765_preDSB[r]' created\n",
      "unknown metabolite 'P02765-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P02765_DSB[r]' created\n",
      "unknown metabolite 'P02765_preNG[r]' created\n",
      "unknown metabolite 'P02765-G3M9[r]' created\n",
      "unknown metabolite 'P02765-M9-misfold[r]' created\n",
      "unknown metabolite 'P02765-M8C-misfold[r]' created\n",
      "unknown metabolite 'P02765-M7A-misfold[r]' created\n",
      "unknown metabolite 'P02765-M6-misfold[r]' created\n",
      "unknown metabolite 'P02765-M5-misfold[r]' created\n",
      "unknown metabolite 'P02765-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P02765-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P02765-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P02765-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P02765-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P02765-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P02765-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P02765-G2M9[r]' created\n",
      "unknown metabolite 'P02765-preG1M9[r]' created\n",
      "unknown metabolite 'P02765-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P02765-G1M9[r]' created\n",
      "unknown metabolite 'P02765-M9[r]' created\n",
      "unknown metabolite 'P02765-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P02765-M8B[r]' created\n",
      "unknown metabolite 'P02765-M8B[g]' created\n",
      "unknown metabolite 'P02765-M7[g]' created\n",
      "unknown metabolite 'P02765-M6[g]' created\n",
      "unknown metabolite 'P02765-M5[g]' created\n",
      "unknown metabolite 'P02765-M5-GN[g]' created\n",
      "unknown metabolite 'P02765-M4-GN[g]' created\n",
      "unknown metabolite 'P02765-M3-GN[g]' created\n",
      "unknown metabolite 'P02765-M3-GN2[g]' created\n",
      "unknown metabolite 'P02765-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P02765-COPII-coated[c]' created\n",
      "unknown metabolite 'P02765-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P02765_preOG[g]' created\n",
      "unknown metabolite 'P02765-acgal[g]' created\n",
      "unknown metabolite 'P02765-Core1[g]' created\n",
      "unknown metabolite 'P02765-Core2[g]' created\n",
      "unknown metabolite 'P02765-preSV[g]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P02765-AP-coated[c]' created\n",
      "unknown metabolite 'P02765_mature[sv]' created\n",
      "unknown metabolite 'P02765_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P02647[c]' created\n",
      "unknown metabolite 'P02647-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P02647-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P02647-BiP-adp[r]' created\n",
      "unknown metabolite 'P02647[r]' created\n",
      "unknown metabolite 'P02647-SRP[c]' created\n",
      "unknown metabolite 'P02647-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P02647-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P02647-SEC61C[c]' created\n",
      "unknown metabolite 'P02647-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P02647-sp[r]' created\n",
      "unknown metabolite 'P02647_preNG[r]' created\n",
      "unknown metabolite 'P02647-G3M9[r]' created\n",
      "unknown metabolite 'P02647-M9-misfold[r]' created\n",
      "unknown metabolite 'P02647-M8C-misfold[r]' created\n",
      "unknown metabolite 'P02647-M7A-misfold[r]' created\n",
      "unknown metabolite 'P02647-M6-misfold[r]' created\n",
      "unknown metabolite 'P02647-M5-misfold[r]' created\n",
      "unknown metabolite 'P02647-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P02647-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P02647-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P02647-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P02647-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P02647-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P02647-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P02647-G2M9[r]' created\n",
      "unknown metabolite 'P02647-preG1M9[r]' created\n",
      "unknown metabolite 'P02647-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P02647-G1M9[r]' created\n",
      "unknown metabolite 'P02647-M9[r]' created\n",
      "unknown metabolite 'P02647-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P02647-M8B[r]' created\n",
      "unknown metabolite 'P02647-M8B[g]' created\n",
      "unknown metabolite 'P02647-M7[g]' created\n",
      "unknown metabolite 'P02647-M6[g]' created\n",
      "unknown metabolite 'P02647-M5[g]' created\n",
      "unknown metabolite 'P02647-M5-GN[g]' created\n",
      "unknown metabolite 'P02647-M4-GN[g]' created\n",
      "unknown metabolite 'P02647-M3-GN[g]' created\n",
      "unknown metabolite 'P02647-M3-GN2[g]' created\n",
      "unknown metabolite 'P02647-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P02647-COPII-coated[c]' created\n",
      "unknown metabolite 'P02647-COPII-uncoated[c]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P02647-preSV[g]' created\n",
      "unknown metabolite 'P02647-AP-coated[c]' created\n",
      "unknown metabolite 'P02647_mature[sv]' created\n",
      "unknown metabolite 'P02647_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P02787[c]' created\n",
      "unknown metabolite 'P02787-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P02787-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P02787-BiP-adp[r]' created\n",
      "unknown metabolite 'P02787[r]' created\n",
      "unknown metabolite 'P02787-SRP[c]' created\n",
      "unknown metabolite 'P02787-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P02787-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P02787-SEC61C[c]' created\n",
      "unknown metabolite 'P02787-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P02787-sp[r]' created\n",
      "unknown metabolite 'P02787_preDSB[r]' created\n",
      "unknown metabolite 'P02787-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P02787_DSB[r]' created\n",
      "unknown metabolite 'P02787_preNG[r]' created\n",
      "unknown metabolite 'P02787-G3M9[r]' created\n",
      "unknown metabolite 'P02787-M9-misfold[r]' created\n",
      "unknown metabolite 'P02787-M8C-misfold[r]' created\n",
      "unknown metabolite 'P02787-M7A-misfold[r]' created\n",
      "unknown metabolite 'P02787-M6-misfold[r]' created\n",
      "unknown metabolite 'P02787-M5-misfold[r]' created\n",
      "unknown metabolite 'P02787-M5-unfold-OS9-BiP-adp-HSP90B1[r]' created\n",
      "unknown metabolite 'P02787-M5-unfold-OS9-SEL1[r]' created\n",
      "unknown metabolite 'P02787-M5-unfold-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P02787-M5-unfold-UBIQP-SEL1-SYVN1[r]' created\n",
      "unknown metabolite 'P02787-M5-unfold-UBIQP[c]' created\n",
      "unknown metabolite 'P02787-unfold-UBIQP-RAD23A[c]' created\n",
      "unknown metabolite 'P02787-unfold-Proteasome[c]' created\n",
      "unknown metabolite 'P02787-G2M9[r]' created\n",
      "unknown metabolite 'P02787-preG1M9[r]' created\n",
      "unknown metabolite 'P02787-G1M9-CALR-CANX[r]' created\n",
      "unknown metabolite 'P02787-G1M9[r]' created\n",
      "unknown metabolite 'P02787-M9[r]' created\n",
      "unknown metabolite 'P02787-M9-UGGT1[r]' created\n",
      "unknown metabolite 'P02787-M8B[r]' created\n",
      "unknown metabolite 'P02787-M8B[g]' created\n",
      "unknown metabolite 'P02787-M7[g]' created\n",
      "unknown metabolite 'P02787-M6[g]' created\n",
      "unknown metabolite 'P02787-M5[g]' created\n",
      "unknown metabolite 'P02787-M5-GN[g]' created\n",
      "unknown metabolite 'P02787-M4-GN[g]' created\n",
      "unknown metabolite 'P02787-M3-GN[g]' created\n",
      "unknown metabolite 'P02787-M3-GN2[g]' created\n",
      "unknown metabolite 'P02787-M8B-LMAN1L[r]' created\n",
      "unknown metabolite 'P02787-COPII-coated[c]' created\n",
      "unknown metabolite 'P02787-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P02787_preOG[g]' created\n",
      "unknown metabolite 'P02787-acgal[g]' created\n",
      "unknown metabolite 'P02787-Core1[g]' created\n",
      "unknown metabolite 'P02787-Core2[g]' created\n",
      "unknown metabolite 'P02787-preSV[g]' created\n",
      "unknown metabolite 'P02787-AP-coated[c]' created\n",
      "unknown metabolite 'P02787_mature[sv]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P02787_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n",
      "Ignoring reaction 'P02787_TRANSLATION_protein' since it already exists.\n",
      "Ignoring reaction 'P02787_post_TRANSLOC_3B_1' since it already exists.\n",
      "Ignoring reaction 'P02787_post_TRANSLOC_3B_2' since it already exists.\n",
      "Ignoring reaction 'P02787_post_TRANSLOC_3B_3' since it already exists.\n",
      "Ignoring reaction 'P02787_co_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'P02787_co_TRANSLOC_2' since it already exists.\n",
      "Ignoring reaction 'P02787_co_TRANSLOC_3' since it already exists.\n",
      "Ignoring reaction 'P02787_co_TRANSLOC_4' since it already exists.\n",
      "Ignoring reaction 'P02787_co_TRANSLOC_5' since it already exists.\n",
      "Ignoring reaction 'P02787_co_TRANSLOC_6' since it already exists.\n",
      "Ignoring reaction 'P02787_BiP_release' since it already exists.\n",
      "Ignoring reaction 'P02787_SP_degradation' since it already exists.\n",
      "Ignoring reaction 'P02787_Start_DSB' since it already exists.\n",
      "Ignoring reaction 'P02787_PDI_1' since it already exists.\n",
      "Ignoring reaction 'P02787_PDI_2' since it already exists.\n",
      "Ignoring reaction 'P02787_Start_NG' since it already exists.\n",
      "Ignoring reaction 'P02787_ERNG' since it already exists.\n",
      "Ignoring reaction 'P02787_Mannosidase_1C' since it already exists.\n",
      "Ignoring reaction 'P02787_Mannosidase_1B_misfold' since it already exists.\n",
      "Ignoring reaction 'P02787_Mannosidase_1A_misfold' since it already exists.\n",
      "Ignoring reaction 'P02787_Mannosidase_1D_misfold' since it already exists.\n",
      "Ignoring reaction 'P02787_OS9_recognition' since it already exists.\n",
      "Ignoring reaction 'P02787_OS9_transfer_SYVN1' since it already exists.\n",
      "Ignoring reaction 'P02787_Ubiquitination_lumped' since it already exists.\n",
      "Ignoring reaction 'P02787_retro_TRANSLOC_2' since it already exists.\n",
      "Ignoring reaction 'P02787_Ubiquitination_deglycosylation' since it already exists.\n",
      "Ignoring reaction 'P02787_Ubiquitination_removal' since it already exists.\n",
      "Ignoring reaction 'P02787_Ubiquitination_degradation' since it already exists.\n",
      "Ignoring reaction 'P02787_OS9_binding_SEL1' since it already exists.\n",
      "Ignoring reaction 'P02787_Glucosidase_1' since it already exists.\n",
      "Ignoring reaction 'P02787_Glucosidase_2' since it already exists.\n",
      "Ignoring reaction 'P02787_CALR_CANX_folding' since it already exists.\n",
      "Ignoring reaction 'P02787_CALR_CANX_release' since it already exists.\n",
      "Ignoring reaction 'P02787_Glucosidase_3' since it already exists.\n",
      "Ignoring reaction 'P02787_Glucosidase_3_misfolded' since it already exists.\n",
      "Ignoring reaction 'P02787_UGGT_Refolding_1' since it already exists.\n",
      "Ignoring reaction 'P02787_UGGT_Refolding_2' since it already exists.\n",
      "Ignoring reaction 'P02787_Mannosidase_1B' since it already exists.\n",
      "Ignoring reaction 'P02787_GOLGI_MAN1A' since it already exists.\n",
      "Ignoring reaction 'P02787_GOLGI_MAN1B' since it already exists.\n",
      "Ignoring reaction 'P02787_GOLGI_MAN1C' since it already exists.\n",
      "Ignoring reaction 'P02787_GOLGI_MGAT1' since it already exists.\n",
      "Ignoring reaction 'P02787_GOLGI_MAN2' since it already exists.\n",
      "Ignoring reaction 'P02787_GOLGI_MAN3' since it already exists.\n",
      "Ignoring reaction 'P02787_GOLGI_MGAT2' since it already exists.\n",
      "Ignoring reaction 'P02787_LMAN1_recognition_A' since it already exists.\n",
      "Ignoring reaction 'P02787_COPII_recruit_A' since it already exists.\n",
      "Ignoring reaction 'P02787_COPII_maturation_A' since it already exists.\n",
      "Ignoring reaction 'P02787_COPII_fusion_A' since it already exists.\n",
      "Ignoring reaction 'P02787_Start_OG' since it already exists.\n",
      "Ignoring reaction 'P02787_GOLGI_O_Gal' since it already exists.\n",
      "Ignoring reaction 'P02787_GOLGI_O_Core1' since it already exists.\n",
      "Ignoring reaction 'P02787_GOLGI_O_Core2' since it already exists.\n",
      "Ignoring reaction 'P02787_Start_Secretion' since it already exists.\n",
      "Ignoring reaction 'P02787_SECRETION_1' since it already exists.\n",
      "Ignoring reaction 'P02787_SECRETION_2' since it already exists.\n",
      "Ignoring reaction 'P02787_Final_location_[e]' since it already exists.\n",
      "Ignoring reaction 'P02787_Final_demand' since it already exists.\n",
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P02741[c]' created\n",
      "unknown metabolite 'P02741-ASNA1-atp[c]' created\n",
      "unknown metabolite 'P02741-ASNA1-atp-SEC61C[r]' created\n",
      "unknown metabolite 'P02741-BiP-adp[r]' created\n",
      "unknown metabolite 'P02741[r]' created\n",
      "unknown metabolite 'P02741-SRP[c]' created\n",
      "unknown metabolite 'P02741-SRP-SRPR-RRBP1[c]' created\n",
      "unknown metabolite 'P02741-SRP-SRPR-SEC61C[c]' created\n",
      "unknown metabolite 'P02741-SEC61C[c]' created\n",
      "unknown metabolite 'P02741-SEC61-SPC[r]' created\n",
      "unknown metabolite 'P02741-sp[r]' created\n",
      "unknown metabolite 'P02741_preDSB[r]' created\n",
      "unknown metabolite 'P02741-ds-P4HB-ox[r]' created\n",
      "unknown metabolite 'P02741_DSB[r]' created\n",
      "unknown metabolite 'P02741-LMAN1L[r]' created\n",
      "unknown metabolite 'P02741-COPII-coated[c]' created\n",
      "unknown metabolite 'P02741-COPII-uncoated[c]' created\n",
      "unknown metabolite 'P02741[g]' created\n",
      "unknown metabolite 'P02741-preSV[g]' created\n",
      "unknown metabolite 'P02741-AP-coated[c]' created\n",
      "unknown metabolite 'P02741_mature[sv]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'SEC61C' since it already exists.\n",
      "Ignoring reaction 'TRAP' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'ARF1_activation' since it already exists.\n",
      "Ignoring reaction 'ARF1_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'SAR1B_activation' since it already exists.\n",
      "Ignoring reaction 'SAR1B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_binding' since it already exists.\n",
      "Ignoring reaction 'RAB1A_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_gdp_degradation' since it already exists.\n",
      "Ignoring reaction 'RAB1A_activation' since it already exists.\n",
      "Ignoring reaction 'RAB6B_activation' since it already exists.\n",
      "Ignoring reaction 'RAB8B_activation' since it already exists.\n",
      "Ignoring reaction 'BiP_NEF' since it already exists.\n",
      "Ignoring reaction 'BiP_ATPase' since it already exists.\n",
      "Ignoring reaction 'BiP_atp_formation' since it already exists.\n",
      "Ignoring reaction 'BiP_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'GOLGI_TO_ENDOSOME_3' since it already exists.\n",
      "Ignoring reaction 'COPI_recycling' since it already exists.\n",
      "Ignoring reaction 'COPII_recycling' since it already exists.\n",
      "Ignoring reaction 'NSF-dissociation' since it already exists.\n",
      "Ignoring reaction 'retro_TRANSLOC_1' since it already exists.\n",
      "Ignoring reaction 'Proteasome_complex' since it already exists.\n",
      "Ignoring reaction 'ASNA1_atp' since it already exists.\n",
      "Ignoring reaction 'ASNA1_adp_degradation' since it already exists.\n",
      "Ignoring reaction 'ASNA1_formation' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2' since it already exists.\n",
      "Ignoring reaction 'CALR_Ca2_degradation' since it already exists.\n",
      "Ignoring reaction 'GTHOX_CtoR_transport' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_GSSG' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1LB' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_H2O2' since it already exists.\n",
      "Ignoring reaction 'ERO1LB_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'ERO1A_reoxidation_1' since it already exists.\n",
      "Ignoring reaction 'PDI_reoxidation_ERO1A' since it already exists.\n",
      "Ignoring reaction 'EXOCYST_COMPLEX' since it already exists.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "unknown metabolite 'P02741_mature[e]' created\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Ignoring reaction 'EXOCYST_degradation' since it already exists.\n",
      "Ignoring reaction 'SECRETION_3' since it already exists.\n",
      "Ignoring reaction 'SRP' since it already exists.\n",
      "Ignoring reaction 'SRPR' since it already exists.\n",
      "Ignoring reaction 'SPC' since it already exists.\n",
      "Ignoring reaction 'SPC_degradation' since it already exists.\n",
      "Ignoring reaction 'co_TRANSLOC_7' since it already exists.\n",
      "Ignoring reaction 'TRAP_degradation' since it already exists.\n",
      "Ignoring reaction 'OST_complex' since it already exists.\n",
      "Ignoring reaction 'Proteasome_degradation' since it already exists.\n",
      "Ignoring reaction 'CAATPaser' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'g3m8mpdol_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_L_to_canonical' since it already exists.\n",
      "Ignoring reaction 'doldp_U_to_canonical' since it already exists.\n",
      "Ignoring reaction 'PROTA_formation' since it already exists.\n",
      "Ignoring reaction 'PROTB_formation' since it already exists.\n",
      "Ignoring reaction 'H2O2ter' since it already exists.\n"
     ]
    }
   ],
   "source": [
    "entryIDs = [\"P60568\",\"P01589\",\"P14784\",\"P31785\",\"P05231\",\"P08887\",\"P40189\",\"P13232\",\"P16871\",\"P01579\",\"P01584\",\"P18510\",\n",
    "            \"P05112\",\"P05113\",\"P10145\",\"P15248\",\"P22301\",\"P29459\",\"P29460\",\"P35225\",\"P40933\",\"Q96F46\",\"Q9Y2Y9\",\"P09919\",\n",
    "            \"P02778\",\"P13500\",\"P10147\",\"P01375\",\"P15692\",\"P01127\",\"P13236\",\"P04141\",\"P09038\",\"P51671\",\"P78556\",\"Q969D9\",\n",
    "           \"Q06830\",\"P32119\",\"Q16552\",\"Q96PD4\",\"Q8NAC3\",\"P01009\",\"P02763\",\"P00738\",\"P01011\",\"P02765\",\"P02647\",\"P02787\",\n",
    "           \"P02787\",\"P02741\"]\n",
    "\n",
    "# Initialize new model\n",
    "secretory_model = model.copy()\n",
    "    \n",
    "for j in range(len(entryIDs)):\n",
    "    # Generate lists for reactions (rxns), reaction names (rxnNames), Gene-Protein-Reactions (GPRs), and pathways (subSystems)\n",
    "    [rxns, rxnNames, GPRs, subSystems] = generateProteinSpecificRxns_A_modified(entryIDs[j])\n",
    "    # Add new reactions to metabolic model\n",
    "    for i in range(len(rxns)):\n",
    "        # Create place holder for creating reactions\n",
    "        r = secretory_model.reactions[0].copy()\n",
    "        r.name = rxnNames[i]\n",
    "        r.id = rxnNames[i]\n",
    "        r.gene_reaction_rule = GPRs[i]\n",
    "        r.subsystem = subSystems[i]\n",
    "        secretory_model.add_reaction(r)\n",
    "        if r._model is not None:\n",
    "            r.build_reaction_from_string(reaction_str=rxns[i],rev_arrow='<-',fwd_arrow='->',reversible_arrow='<=>')\n",
    "    \n",
    "cobra_io_mat.save_matlab_model(secretory_model, 'recon2.2s_sarscov2.mat')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.10"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
